fix: schedule subnav page translations

emmambd opened the pull request:
This PR includes translations for submenu pages under Examples in https://gtfs.org/schedule/examples/translations/, including: 

- Example GTFS Feed
- Fares v1
- Fares v2
- Frequencies
- Pathways
- Translations
- Routes, stops, trips
- Continuous stops
- Text to speech
- Attributions
- Feed info

As well as the Changes and Validate pages.

It should be noted there seems to be recurring issues with Simpleen in deleting links (maybe an issue with markdown?) and not honouring glossary do not translate items, particularly for languages without the Latin alphabet. These have been resolved manually.

emmambd committed:
add glossaries to CLI config

emmambd committed:
GTFS Schedule example feed translations

emmambd committed:
exclude translations from search

emmambd committed:
fares v1 examples

emmambd committed:
fares v2 example translations

emmambd committed:
rename translation files fares v2

emmambd committed:
frequencies example translations

emmambd committed:
pathways example translations

emmambd committed:
translations.txt examples translations

derhuerst commented on docs/schedule/example-feed.de.md:
```diff
@@ -0,0 +1,178 @@
+## translations.txt
+
+    table_name,field_name,language,translation,record_id
+    stops,stop_name,en,Tokyo Station,stopid000001
```
Unless I didn't understand the note about the "files not being connected", this stop doesn't appear in `stops.txt` above.

emmambd replied:
I believe each `.txt` example is different and unrelated to each other.

github-actions[bot] committed:
[create-pull-request] automated change (#67)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

github-actions[bot] committed:
[create-pull-request] automated change (#68)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
routes stops trips examples translations

emmambd committed:
continuous stop examples translations

emmambd committed:
feed info example translations

emmambd committed:
attribution example translations

emmambd committed:
examples index translations

emmambd committed:
remove edit link from translated pages

emmambd committed:
example index translations

emmambd committed:
changes index translations

emmambd committed:
GTFS schedule processes translations

github-actions[bot] committed:
[create-pull-request] automated change (#70)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
feat: Add clearer contribution info (#69)

* update contributing information to clarity how translations work

* applying feedback

* about translations

emmambd committed:
feat: Add clearer contribution info (#69)

* update contributing information to clarity how translations work

* applying feedback

* about translations

github-actions[bot] committed:
[create-pull-request] automated change (#71)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
update home pages traditional chinese

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/MobilityData/gtfs.org?pullRequest=66) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/MobilityData/gtfs.org?pullRequest=66) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: emmambd<br/>:white_check_mark: briandonahue<br/>:x: github-actions[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/MobilityData/gtfs.org?pullRequest=66) it.</sub>

emmambd committed:
validate page translations

emmambd committed:
Merge branch 'main' into schedule_examples

github-actions[bot] committed:
[create-pull-request] automated change (#81)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
fix: update docs on translation contributions (#85)

We want to welcome translation contributions, while being clear that any contributions may be subject to being overwritten in the future when there are documentation changes.

josee-sabourin commented on docs/index.zh_TW.md:
```diff
@@ -3,15 +3,14 @@ search:
 </div>
 
-这General Transit Feed Specification (GTFS) 是一种数据规范，允许公共交通机构以[多种软件应用程序](resources/apps)可以使用的格式发布其交通数据。今天，GTFS数以千计的公共交通提供商使用数据格式。
+這General Transit Feed Specification(GTFS ) 是一種數據規範，允許公共交通機構以可被[各種軟件應用程序](resources/apps)使用的格式發布其交通數據。今天，GTFS數以千計的公共交通提供商使用數據格式。
```
Is the GTFS Realtime button meant to be translated?

josee-sabourin commented on docs/index.zh_TW.md:
```diff
@@ -3,15 +3,14 @@ search:
-GTFS被分成一个Schedule包含的[组件](schedule)Schedule、票价和地理交通信息以及包含到达预测、车辆位置和服务建议[的实时组件](realtime)。
-
-[更多的GTFS背景](background.md)
+[更多的](background.md)GTFS[背景](background.md)
```
Is this hyperlink meant to be broken up like this? Or should GTFS be included.

emmambd replied:
It's structured the same way in the original English version: https://github.com/MobilityData/gtfs.org/edit/main/docs/index.md

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+
+Um zu GTFS beizutragen, lesen Sie den [Specification Amendment Process](../process) und verfolgen Sie die Diskussionen in den offenen [Fragen](https://github.com/google/transit/issues) und [Pull Requests](https://github.com/google/transit/pulls) auf Googles Transit Github Repository[(google/transit](https://github.com/google/transit)). ![](../../assets/mark-github.svg)
+
+!!!note ""
```
this appears in the viewer, I'm unsure if it will disappear once rendered?

emmambd replied:
It disappears once rendered, but I can do some checks

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+
+Die GTFS ist nicht in Stein gemeißelt. Vielmehr handelt es sich um eine offene Spezifikation, die von der Gemeinschaft der Verkehrsunternehmen, Entwickler und anderen Beteiligten, die GTFS verwenden, entwickelt und gepflegt wird. Es wird erwartet, dass diese Gemeinschaft von Produzenten und Nutzern von GTFS Vorschläge zur Erweiterung der Spezifikation unterbreiten wird, um neue Funktionen zu ermö ...
+
+Um zu GTFS beizutragen, lesen Sie den [Specification Amendment Process](../process) und verfolgen Sie die Diskussionen in den offenen [Fragen](https://github.com/google/transit/issues) und [Pull Requests](https://github.com/google/transit/pulls) auf Googles Transit Github Repository[(google/transit](https://github.com/google/transit)). ![](../../assets/mark-github.svg)
```
Issues translated but Pull Request not?

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+    <div class="featurelist">
+        <ul>
+            <li>Fügt neue <code>transfer_type</code>'s für trip hinzu, um zu definieren, ob ein Benutzer einen "In-Seat-Transfer" durchführen kann, wenn dasselbe vehicle zwei aufeinanderfolgende Fahrten durchführt und der Benutzer an Bord bleiben kann</li>
+            <li>Es kann definiert werden, wann Transfers auf dem Sitzplatz nicht erlaubt sind, aber zwei verschiedene Fahrten betrieblich miteinander verbunden werden können.
```
"in seat transfer" becomes "In-Seat-Transfer" in line 54, but gets translated in 55

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">Hinzufügen von trip mit Sitzplatzoption</a></h3>
```
PR title gets translated here, not sure if that's meant to happen

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">GTFS Fares V2 Basis-Implementierung</a></h3>
```
PR title translated

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">trip und Strecke-zu-Strecke-Transfers</a></h3>
```
PR title translated

josee-sabourin commented on docs/schedule/changes/index.de.md:
```diff
@@ -0,0 +1,112 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">Hinzufügen von trip mit Sitzplatzoption</a></h3>
+        <p class="maintainer">#303 von <a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a> wurde zusammengeführt am 26. Juni 2022</p>
```
Date is written differently in this PR than the others "zusammengeführt" appearing before the date.

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+
+Para contribuir GTFS, lea el [proceso de modificación de la especificación](../process) y siga los debates en las [cuestiones](https://github.com/google/transit/issues) abiertas y las [solicitudes de extracción](https://github.com/google/transit/pulls) en el repositorio Github de Google Transit[(google/transit](https://github.com/google/transit)). ![](../../assets/mark-github.svg)
+
+Nota ""
```
same as Dutch translation, this appears in the viewier

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">Añadir transferencias trip con opción de asiento</a></h3>
+        <p class="maintainer">#303 by <a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a> was merged on Jun 26, 2022</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Implementación base deGTFSFares V2</a></h3>
```
nit: space missing between "de GTFS-Fares", hyphen missing between "GTFS-Fares"

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Implementación base deGTFSFares V2</a></h3>
+        <p class="maintainer">#286 by <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> was merged on Puede 17, 2022</p>
```
half translated

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">Transferencias detrip y de ruta a ruta</a></h3>
```
nit: space needed between "de trip"

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">Transferencias detrip y de ruta a ruta</a></h3>
+        <p class="maintainer">#284 by <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> was merged on Oct 5, 2021</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.es.md:
```diff
@@ -0,0 +1,111 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/208" class="no-icon" target="_blank">GTFS-ContinuousStops</a></h3>
+        <p class="maintainer">#208 by <a href="https://github.com/timMillet" class="no-icon" target="_blank">timMillet</a> was merged on Puede 13, 2020</p>
```
half translated

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+  exclude: true
+---
+
+# Modifications duGTFS Schedule
```
nit: need a space between "du GTFS"

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+
+Pour contribuer à GTFS, lisez le [processus de modification des spécifications](../process) et suivez les discussions dans les [questions](https://github.com/google/transit/issues) ouvertes et les [demandes de retrait](https://github.com/google/transit/pulls) sur le dépôt Github Transit de Google[(google/transit](https://github.com/google/transit)). ![](../../assets/mark-github.svg)
+
+! !! note ""
```
same as other translations

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+
+## Propositions récemment fusionnées![](../../assets/pr-merged.svg)
+
+Propositions récemment fusionnées qui sont maintenant des caractéristiques de la [spécification officielle](../reference) du [GTFS Schedule](../reference). Consultez l'[historique](../process#revision-history) complet [des révisions](../process#revision-history) pour en savoir plus.
```
both hyperlinks are fragmented, 1st  by "du", 2nd by "complet"

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Implémentation de la baseGTFSFares V2</a></h3>
+        <p class="maintainer">#Le numéro 286 de <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> a été fusionné le Peut 17, 2022</p>
```
translating the month of May as the verb may. This may have also happened in the Spanish translations as well (my Spanish is rusty which I why I didn't point it out)

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Implémentation de la baseGTFSFares V2</a></h3>
```
need spaces and hypens in PR title

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/208" class="no-icon" target="_blank">GTFS-ContinuousStops</a></h3>
+        <p class="maintainer">#208 par <a href="https://github.com/timMillet" class="no-icon" target="_blank">timMillet</a> a été fusionné le Peut 13, 2020</p>
```
same May issue here

josee-sabourin commented on docs/schedule/changes/index.fr.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">Transferts detrip et d'itinéraire à itinéraire</a></h3>
```
nit: space missing between "de trip"

josee-sabourin commented on docs/schedule/changes/index.id.md:
```diff
@@ -0,0 +1,57 @@
+
+Itu GTFS Spesifikasi tidak diatur dalam batu. Sebaliknya, ini adalah spesifikasi terbuka yang dikembangkan dan dikelola oleh komunitas agen transit, pengembang, dan pemangku kepentingan lain yang menggunakannya GTFS . Diharapkan komunitas produsen dan konsumen ini GTFS data akan memiliki proposal untuk memperluas spesifikasi guna mengaktifkan kemampuan baru.
+
+Untuk berkontribusi pada GTFS , baca [Proses Amendemen Spesifikasi](../process) dan ikuti diskusi dalam [masalah](https://github.com/google/transit/issues) terbuka dan [tarik permintaan](https://github.com/google/transit/pulls) di repositori Transit Github Google ( [google/transit](https://github.com/google/transit) ).![](../../assets/mark-github.svg)
```
nit: errant spaces in google/transit hyperlink

josee-sabourin commented on docs/schedule/changes/index.id.md:
```diff
@@ -0,0 +1,57 @@
+
+Untuk berkontribusi pada GTFS , baca [Proses Amendemen Spesifikasi](../process) dan ikuti diskusi dalam [masalah](https://github.com/google/transit/issues) terbuka dan [tarik permintaan](https://github.com/google/transit/pulls) di repositori Transit Github Google ( [google/transit](https://github.com/google/transit) ).![](../../assets/mark-github.svg)
+
+!!! catatan ""
```
same issue as previous translations

josee-sabourin commented on docs/schedule/changes/index.id.md:
```diff
@@ -0,0 +1,57 @@
+
+## Proposal yang Baru Digabung![](../../assets/pr-merged.svg)
+
+Baru-baru ini menggabungkan proposal yang sekarang menjadi fitur [resmi](../reference) GTFS Schedule [spesifikasi](../reference) . Lihat [Riwayat Revisi](../process#revision-history) lengkap untuk lebih lanjut.
```
hyperlink fragmented

josee-sabourin commented on docs/schedule/changes/index.id.md:
```diff
@@ -0,0 +1,57 @@
+
+<div class="row"><div class="leftcontainer"><h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank"> Menambahkan-trip-ke-trip transfer dengan opsi di kursi</a></h3><p class="maintainer"> #303 oleh <a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a> digabungkan pada 26 Juni 2022</p></div><div class="featurelist"><ul><li> Menam ...
+
+<div class="row"><div class="leftcontainer"><h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank"> GTFS Fares V2 implementasi basis</a></h3><p class="maintainer"> #286 oleh <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> digabungkan pada 17 Mei 2022</p></div><div class="featurelist"><ul><li> Tarif transit dan tiket</l ...
```
nit: missing hyphen for GTFS-Fares

josee-sabourin commented on docs/schedule/changes/index.ja.md:
```diff
@@ -0,0 +1,113 @@
+
+GTFSに貢献するには、[仕様修正](../process)プロセスを読み、GoogleのTransit Githubリポジトリ[（google/transit](https://github.com/google/transit)）の[オープンイシューや](https://github.com/google/transit/issues) [プルリクエストで](https://github.com/google/transit/pulls)議論に従ってください。![](../../assets/mark-github.svg)
+
+!!!注意 ""
```
I'm starting to think this is meant to be here

josee-sabourin commented on docs/schedule/changes/index.ja.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/208" class="no-icon" target="_blank">GTFS-ContinuousStops（連続停車駅</a></h3>
```
not sure where "（連続停車駅 " comes from in the PR title.

josee-sabourin commented on docs/schedule/changes/index.ja.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/208" class="no-icon" target="_blank">GTFS-ContinuousStops（連続停車駅</a></h3>
+        <p class="maintainer">#208 by<a href="https://github.com/timMillet" class="no-icon" target="_blank">timMillet</a>は2020年5月13日にマージされました。</p>
```
this appears different from the other PRs

josee-sabourin commented on docs/schedule/changes/index.ko.md:
```diff
@@ -0,0 +1,58 @@
+
+기여하다GTFS , [사양 수정 프로세스](../process) 를 읽고 Google의 Transit Github 저장소( [google/transit](https://github.com/google/transit) )에서 열린 [문제](https://github.com/google/transit/issues) 및 [풀 요청](https://github.com/google/transit/pulls) 의 토론을 따르십시오.![](../../assets/mark-github.svg)
+
+!!! 노트 ""
```
i'll stop commenting on these now...

josee-sabourin commented on docs/schedule/changes/index.ko.md:
```diff
@@ -0,0 +1,58 @@
+
+## 최근 병합된 제안서![](../../assets/pr-merged.svg)
+
+이제 [공식](../reference) 의 기능이 된 최근에 병합된 제안GTFS Schedule [사양](../reference) . 자세한 내용은 전체 [개정 내역](../process#revision-history) 을 참조하십시오.
```
fragmented hyperlink

josee-sabourin commented on docs/schedule/changes/index.ko.md:
```diff
@@ -0,0 +1,58 @@
+
+<div class="row"><div class="leftcontainer"><h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">추가하다trip -에게-trip 좌석 옵션이 있는 환승</a></h3><p class="maintainer"> #303 by <a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a> 는 2022년 6월 26일에 합병되었습니다.</p></div><div class="featurelist"><ul><li> 다음에 대한 새로운 <code>transfer_type</code> ...
+
+<div class="row"><div class="leftcontainer"><h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank"> GTFS Fares V2 기본 구현</a></h3><p class="maintainer"> #286 by <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> 가 2022년 5월 17일에 합병되었습니다.</p></div><div class="featurelist"><ul><li> 대중교통 요금 및 티켓</li><li> 복잡한 운임 및 환승 비용 모델링(다중 네 ...
```
nit: missing hyphen between GTFS-Fares

josee-sabourin commented on docs/schedule/changes/index.pt_BR.md:
```diff
@@ -0,0 +1,113 @@
+
+A Especificação GTFS não é definida em pedra. Ao invés disso, é uma especificação aberta desenvolvida e mantida pela comunidade de agências de trânsito, desenvolvedores e outras partes interessadas que utilizam GTFS. Espera-se que esta comunidade de produtores e consumidores de dados GTFS tenha propostas para ampliar as especificações a fim de permitir novas capacidades.
+
+Para contribuir com o GTFS, leia o [Processo de Emenda de Especificações](../process) e siga as discussões nas [questões](https://github.com/google/transit/issues) em aberto e [faça solicitações](https://github.com/google/transit/pulls) no repositório Transit Github do Google[(google/transit](https://github.com/google/transit)). ![](../../assets/mark-github.svg)
```
nit: need space between "Transit Github do Google" and "(google/transit)"

josee-sabourin commented on docs/schedule/changes/index.pt_BR.md:
```diff
@@ -0,0 +1,113 @@
+
+## Propostas fundidas recentemente![](../../assets/pr-merged.svg)
+
+Recentemente foram fundidas propostas que agora são características da [especificação oficial](../reference) da [GTFS Schedule](../reference). Veja o [Histórico de Revisões](../process#revision-history) completo para mais informações.
```
fragmented hyperlink

josee-sabourin commented on docs/schedule/changes/index.pt_BR.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Implementação da base GTFS Fares V2</a></h3>
```
nit: hyphen needed in GTFS-Fares

josee-sabourin commented on docs/schedule/changes/index.pt_BR.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">Transferências detrip e de rota para rota</a></h3>
```
nit: space needed between "de trip"

josee-sabourin commented on docs/schedule/changes/index.pt_BR.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">Transferências detrip e de rota para rota</a></h3>
+        <p class="maintainer">#284 by <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> foi fundido em 5 de outubro de 2021</p>
```
this is translated different than the others

josee-sabourin commented on docs/schedule/changes/index.pt_BR.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/180" class="no-icon" target="_blank">GTFS-Translations</a></h3>
+        <p class="maintainer">#180 pela <a href="https://github.com/LeoFrachet" class="no-icon" target="_blank">LeoFrachet</a> foi fundida em 9 de janeiro de 2020</p>
```
this is translated differently than the others

josee-sabourin commented on docs/schedule/changes/index.ru.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">Добавить пересадку с trip с возможностью пересадки на сиденье</a></h3>
+        <p class="maintainer">#303 by <a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a> was merged on Jun 26, 2022</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.ru.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Базовая реализация GTFSFares V2</a></h3>
+        <p class="maintainer">#286 by <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> was merged on May 17, 2022</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.ru.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/284" class="no-icon" target="_blank">пересадки отtrip и от маршрута к маршруту</a></h3>
+        <p class="maintainer">#284 by <a href="https://github.com/scmcca" class="no-icon" target="_blank">scmcca</a> was merged on Oct 5, 2021</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.ru.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/208" class="no-icon" target="_blank">GTFS-ContinuousStops</a></h3>
+        <p class="maintainer">#208 by <a href="https://github.com/timMillet" class="no-icon" target="_blank">timMillet</a> was merged on May 13, 2020</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.ru.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">Базовая реализация GTFSFares V2</a></h3>
```
nit: hyphen needed in GTFS-Fares

josee-sabourin commented on docs/schedule/changes/index.zh.md:
```diff
@@ -0,0 +1,113 @@
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">增加trip转接，带座位选项</a></h3>
+        <p class="maintainer">#303 by<a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a>was merged on Jun 26, 2022</p>
```
did not get translated

josee-sabourin commented on docs/schedule/changes/index.zh.md:
```diff
@@ -0,0 +1,113 @@
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/286" class="no-icon" target="_blank">GTFS Fares V2基础实现</a></h3>
```
nit: missing hyphen in GTFS-Fares

josee-sabourin commented on docs/schedule/changes/index.zh_TW.md:
```diff
@@ -0,0 +1,58 @@
+
+！！！筆記 ””
+
+  官方規範、參考和文檔均以英文編寫。如果一個Translation不同的語言與英文原文不同，後者優先。所有交流均以英語進行。
```
this does not look like it formatted properly

josee-sabourin commented on docs/schedule/changes/index.zh_TW.md:
```diff
@@ -0,0 +1,58 @@
+
+## 最近合併的提案![](../../assets/pr-merged.svg)
+
+最近合併的提案現在是[官方](../reference)的功能GTFS Schedule[規範](../reference)。有關更多信息，請參閱完整的[修訂歷史](../process#revision-history)。
```
fragmented hyperlink

josee-sabourin commented on docs/schedule/changes/index.zh_TW.md:
```diff
@@ -0,0 +1,58 @@
+
+最近合併的提案現在是[官方](../reference)的功能GTFS Schedule[規範](../reference)。有關更多信息，請參閱完整的[修訂歷史](../process#revision-history)。
+
+<div class="row"><div class="leftcontainer"><h3 class="title"><a href="https://github.com/google/transit/pull/303" class="no-icon" target="_blank">添加trip-至-trip帶座位選項的接送服務</a></h3><p class="maintainer">#303 by <a href="https://github.com/gcamp" class="no-icon" target="_blank">gcamp</a>於 2022 年 6 月 26 日合併</p></div><div class="featurelist"><ul><li>添加新的<code>transfer_type</code> `strip至trip轉移定義用戶是否可以 ...
```
by gcamp did not get translated the same as the others

josee-sabourin commented on docs/schedule/example-feed.de.md:
```diff
@@ -0,0 +1,178 @@
+
+# Beispiel GTFS-Feed
+
+Dieses GTFS-Beispiel zeigt kommagetrennte Datenbeispiele für jede Datei in einem Transit-Feed. Die hier gezeigten Beispieldateien sind nicht alle miteinander verbunden. Sie können auch [einen vollständigen GTFS-Feed](https://github.com/google/transit/blob/master/gtfs/spec/en/examples/sample-feed-1.zip?raw=true) in der endgültigen Form [herunterladen](https://github.com/google/transit/blob/master/ ...
```
fragmented hyperlink

josee-sabourin commented on docs/schedule/example-feed.fr.md:
```diff
@@ -0,0 +1,178 @@
+
+## calendar_dates.txt
+
+Cet exemple montre les exceptions de service pour le jour férié de l'Independence Day en 2006. Le lundi 3 juillet 2006, le service régulier en semaine`(service_id=WD`) est interrompu`(exception_type=2`). À la place, le service de week-end`(service_id=WE`) fonctionne à cette date`(exception_type=1`). Le même changement s'applique également le mardi 4 juillet.
```
nit: code formatting is slightly off, starts a space and a bracket early.

emmambd replied:
@josee-sabourin Could you clarify where this issue starts in the file? I'm having some trouble comparing and contrasting.

josee-sabourin replied:
`(service_id=WD`), `(exception_type=2`), `(service_id=WE`), `(exception_type=1`) should all be 
(`service_id=WD`), (`exception_type=2`), (`service_id=WE`), (`exception_type=1`)

Notice how in the first line, the opening bracket is formatted as code? IIRC the brackets should not be formatted as such. To the point about space, it seems there is a space missing before each of these brackets. 

Hope that helps!

josee-sabourin commented on docs/schedule/example-feed.ja.md:
```diff
@@ -0,0 +1,178 @@
+
+# GTFSフィードの例
+
+このGTFSフィードの例では、トランジットフィード内の各ファイルについて、カンマ区切りでデータサンプルを示しています。ここに示されたサンプルデータファイルは、すべてが互いに関連しているわけではありません。また、最終形式の[完全なGTFSフィードを](https://github.com/google/transit/blob/master/gtfs/spec/en/examples/sample-feed-1.zip?raw=true)ダウンロードして作業することもできますし、同じフィードの個々のファイルをオンラインで閲覧することもできます[。](https://github.com/google/transit/tree/master/gtfs/spec/en/examples/sample-feed-1)
```
seems like the last hyperlink is only on the period

josee-sabourin commented on docs/schedule/example-feed.ja.md:
```diff
@@ -0,0 +1,178 @@
+
+## calendar_dates.txt
+
+この例では、2006年の独立記念日の祝日におけるサービスの例外を示します。2006年7月3日(月)は、平日の通常運行`(service_id=WD`)が中断されます`(exception_type=2`)。代わりに、週末サービス`(service_id=WE`)がその日に運行されます`(exception_type=1`)。7月4日（火）も同様に変更となります。
```
nit: code format starts a little early (includes space and bracket)

josee-sabourin commented on docs/schedule/example-feed.pt_BR.md:
```diff
@@ -0,0 +1,178 @@
+
+## calendar_dates.txt
+
+Este exemplo mostra exceções de serviço para o feriado do Dia da Independência em 2006. Na segunda-feira 3 de julho de 2006, o serviço regular do dia da semana`(service_id=WD`) é interrompido`(exception_type=2`). Em vez disso, o serviço de fim de semana`(service_id=WE`) é interrompido nessa data`(exception_type=1`). A mesma mudança se aplica também na terça-feira 4 de julho.
```
same code formatting nit

josee-sabourin commented on docs/schedule/example-feed.ru.md:
```diff
@@ -0,0 +1,178 @@
+
+## calendar_dates.txt
+
+Этот пример показывает исключения в обслуживании для праздника Дня независимости в 2006 году. В понедельник, 3 июля 2006 года, регулярное обслуживание в будние дни`(service_id=WD`) прерывается`(exception_type=2`). Вместо этого в этот день работает сервис выходного дня`(service_id=WE`)`(exception_type=1`). Это же изменение действует и во вторник 4 июля.
```
same here

josee-sabourin commented on docs/schedule/example-feed.zh.md:
```diff
@@ -0,0 +1,178 @@
+
+## calendar_dates.txt
+
+这个例子显示了2006年独立日假期的服务例外情况。在2006年7月3日星期一，正常的工作日服务`（service_id=WD`）被中断`（exception_type=2`）。相反，周末服务`（service_id=WE`）在该日运行`（exception_type=1`）。同样的变化也适用于7月4日星期二。
```
and here

josee-sabourin commented on docs/schedule/examples/attributions.es.md:
```diff
@@ -0,0 +1,31 @@
+
+Algunos conjuntos de datos GTFS contienen datos agregados de múltiples fuentes, como diferentes proveedores de servicios que sirven a la misma jurisdicción. En algunos casos, es necesario clasificar los organismos que figuran en [agency.txt](../../reference/#agencytxt) como productores, operadores o autoridades.
+
+Por ejemplo, Rejseplanen es un motor de búsqueda de servicios ferroviarios y de autobuses en Dinamarca. La empresa publica un conjunto de datos GTFS que contiene datos de múltiples agencias y proveedores, como se muestra a continuación en agency. [agency.txt](../../reference/#agencytxt).
```
hyperlink formatting issue

josee-sabourin commented on docs/schedule/examples/attributions.es.md:
```diff
@@ -0,0 +1,31 @@
+    206,NT,https://www.nordjyllandstrafikselskab.dk,Europe/Berlin,
+    276,Rejseplanen,https://www.rejseplanen.dk,Europe/Berlin,
+
+Para atribuir a Rejseplanen como productor de datos, se utiliza el archivo [attributions.txt](../../reference/#attributionstxt).txt, donde se define un id atribución junto con el nombre y la url de la organización. Los campos `is_producer`, `is_operator` y `is_authority` se utilizan para categorizar a Rejseplanen como se muestra a continuación:
```
hyperlink formatting issue 
(extra .txt)

josee-sabourin commented on docs/schedule/examples/attributions.id.md:
```diff
@@ -0,0 +1,31 @@
+
+## Mengatribusikan data ke penghasil data secara agregatGTFS Himpunan data
+
+BeberapaGTFS kumpulan data berisi data yang dikumpulkan dari berbagai sumber seperti penyedia layanan berbeda yang melayani yurisdiksi yang sama. Dalam beberapa kasus, perlu untuk mengklasifikasikan lembaga yang terdaftar [agency.txt](../../reference/#agencytxt) sebagai produsen, operator, atau otoritas.
```
nit: missing space between "Beberapa GTFS"

josee-sabourin commented on docs/schedule/examples/attributions.id.md:
```diff
@@ -0,0 +1,31 @@
+
+BeberapaGTFS kumpulan data berisi data yang dikumpulkan dari berbagai sumber seperti penyedia layanan berbeda yang melayani yurisdiksi yang sama. Dalam beberapa kasus, perlu untuk mengklasifikasikan lembaga yang terdaftar [agency.txt](../../reference/#agencytxt) sebagai produsen, operator, atau otoritas.
+
+Misalnya, Rejseplanen adalah mesin pencari untuk layanan kereta api dan bus di Denmark. Perusahaan menerbitkan aGTFS kumpulan data yang berisi data dari beberapa agensi dan penyedia seperti yang ditunjukkan di bawah ini [agency.txt](../../reference/#agencytxt).
```
same here, missing space before GTFS

josee-sabourin commented on docs/schedule/examples/attributions.ja.md:
```diff
@@ -0,0 +1,31 @@
+
+## GTFSデータセットに集約されたデータ生産者への属性データ
+
+GTFSのデータセットには、同じ管轄区域にサービスを提供する異なるサービスプロバイダーなど、複数のソースから集約されたデータを含むものがある。場合によっては、[agency.txtに](../../reference/#agencytxt)記載されている機関を、生産者、運営者、当局のいずれかに分類する必要がある。
```
hyperlink include Japanese character

josee-sabourin commented on docs/schedule/examples/attributions.ja.md:
```diff
@@ -0,0 +1,31 @@
+    206,NT,https://www.nordjyllandstrafikselskab.dk,Europe/Berlin,
+    276,Rejseplanen,https://www.rejseplanen.dk,Europe/Berlin,
+
+Rejseplanenをデータ作成者として特定するために、[attributions.txtという](../../reference/#attributionstxt)ファイルを使用します。このファイルには、組織の名前とURLとともに、attribution IDが定義されています。`is_producer`、`is_operator`、`is_authorityの`各フィールドは、以下のようにRejseplanenを分類するために使用されます。
```
hyperlink include Japanese characters, as does `is_authority`

josee-sabourin commented on docs/schedule/examples/attributions.zh.md:
```diff
@@ -0,0 +1,31 @@
+
+一些GTFS数据集包含从多个来源汇总的数据，如服务于同一管辖区的不同服务提供商。在某些情况下，有必要将[agency.txt](../../reference/#agencytxt)中列出的机构分类为生产者、经营者或当局。
+
+例如，Rejseplanen是丹麦的一个铁路和公共汽车服务搜索引擎。该公司发布了一个GTFS数据集，其中包含来自多个机构和供应商的数据，如下图[agency.txt](../../reference/#agencytxt)txt所示。
```
extra txt

josee-sabourin commented on docs/schedule/examples/continuous-stops.de.md:
```diff
@@ -0,0 +1,71 @@
+    E,Oro Grande Post Office,34.599292,-117.334452
+    F,Silver Lakes Market,34.744662,-117.335407
+
+In [stop_times.txt](../../reference/#stoptimestxt)xt, für eine bestimmte trip:
```
extra xt after hyperlink

josee-sabourin commented on docs/schedule/examples/continuous-stops.es.md:
```diff
@@ -0,0 +1,71 @@
+    E,Oro Grande Post Office,34.599292,-117.334452
+    F,Silver Lakes Market,34.744662,-117.335407
+
+En [stop_times.txt](../../reference/#stoptimestxt).txt, para un trip determinado:
```
extra .txt after hyperlink

josee-sabourin commented on docs/schedule/examples/continuous-stops.ja.md:
```diff
@@ -0,0 +1,71 @@
+
+ビクターバレー・トランジット（米国カリフォルニア州ビクタービル）は、22号線の一部区間に限り、連続停車制度を適用しています。乗客は、County Fareゾーン内の安全な場所でのみバスに乗り降りすることができます。Local Fareゾーンでは、連続したピックアップとドロップオフはできません。
+
+ローカル料金ゾーンとカウンティ料金ゾーンは、下図のようにエア・エクスプレス・ウェイで区切られています。予定停車駅のNational Trails Highway - Air Expresswayは、この境界のやや北側に位置しています。正確には、交通機関は、バスルートと境界線との実際の交差点に停留所を追加し、そこから連続した送迎を行うことができます。この停留所は予定外のままでもよい。
```
I think a couple do not translates got translated here when comparing with other languages

josee-sabourin commented on docs/schedule/examples/continuous-stops.ja.md:
```diff
@@ -0,0 +1,71 @@
+    E,Oro Grande Post Office,34.599292,-117.334452
+    F,Silver Lakes Market,34.744662,-117.335407
+
+[stop_times.txt](../../reference/#stoptimestxt)xtでは、指定されたトリップについて。
```
extra xt after hyperlink

josee-sabourin commented on docs/schedule/examples/continuous-stops.ko.md:
```diff
@@ -0,0 +1,71 @@
+
+## 모든 곳에서 픽업 및 하차
+
+대중교통 기관인 The Current(미국 버몬트주 로킹엄)는 2번, 53번, 55번 노선에 연속 정차 정책을 적용합니다.scheduled 버스가 정차하기에 안전한 장소가 있는 한 경로를 따라 정차합니다.
```
.scheduled ? translation error it seems

josee-sabourin commented on docs/schedule/examples/continuous-stops.pt_BR.md:
```diff
@@ -0,0 +1,71 @@
+
+## Pegar e largar em um trecho do itinerário
+
+A agência de trânsito Victor Valley Transit (Victorville, US-CA) aplica uma política de parada contínua somente em uma parte da rota 22. Um passageiro pode embarcar e descer do ônibus em qualquer local seguro dentro da zona de tarifas do condado somente. Não é possível fazer coletas e descidas contínuas dentro da zona de tarifas locais.
+
+A zona de Tarifa Local e a zona de Tarifa do Município são separadas pela Via Aérea Expressa, como mostrado na figura abaixo. A parada da National Trails Highway - Air Expressway está localizada um pouco ao norte deste limite. Para ser preciso, a agência de trânsito pode acrescentar uma parada no cruzamento real da rota do ônibus com o limite, a partir do qual a coleta e entrega contínua está dis ...
```
do not translates getting translated again

josee-sabourin commented on docs/schedule/examples/continuous-stops.ru.md:
```diff
@@ -0,0 +1,71 @@
+
+## Посадка и высадка везде
+
+Транзитное агентство The Current (Рокингем, США-ВТ) применяет политику непрерывных остановок на маршрутах 2, 53 и 55. Пассажира можно забирать и высаживать между scheduled остановками на всем протяжении маршрута, если есть безопасное место для остановки автобуса.
```
Rockingham, US-VT got translated

josee-sabourin commented on docs/schedule/examples/continuous-stops.ru.md:
```diff
@@ -0,0 +1,71 @@
+
+Транзитное агентство Victor Valley Transit (г. Викторвилл, штат Калифорния) применяет политику непрерывных остановок только на части маршрута 22. Пассажир может садиться и выходить из автобуса в любом безопасном месте только в пределах зоны действия окружного тарифа. Непрерывные посадки и высадки невозможны в зоне местного тарифа.
+
+Зона местного тарифа и зона окружного тарифа разделены скоростной автострадой Air Expressway, как показано на рисунке ниже. scheduled остановка National Trails Highway - Air Expressway расположена немного севернее этой границы. Для точности транзитное агентство может добавить остановку на фактическом пересечении автобусного маршрута с этой границей, с которой возможна непрерывная посадка и высадк ...
```
more do not translates getting translated

josee-sabourin commented on docs/schedule/examples/continuous-stops.zh.md:
```diff
@@ -0,0 +1,71 @@
+
+维克多谷公交公司（Victorville, US-CA）只在22路的一部分实行连续停靠政策。乘客只能在县城收费区的任何安全地点上车和下车。在本地票价区域内，不可能有连续的上下车。
+
+如下图所示，本地票价区和县城票价区被空中高速路隔开。scheduled站点National Trails Highway - Air Expressway位于该边界的稍北处。为了准确起见，公交机构可以在公交线路与边界的实际交汇处增加一个站点，从这里可以连续接送乘客。这个站点可以保持unscheduled。
```
scheduled and unscheduled not translated, other do not translate did get translated (same as other languages)

josee-sabourin commented on docs/schedule/examples/continuous-stops.zh_TW.md:
```diff
@@ -0,0 +1,71 @@
+
+## 隨處接送
+
+公交機構 The Current（羅金厄姆，美國佛蒙特州）對 2、53 和 55 號路線實施連續停車政策。scheduled只要公共汽車有安全的地方停靠，沿途都會停靠。
```
Rockingham, US-VT translated

josee-sabourin commented on docs/schedule/examples/continuous-stops.zh_TW.md:
```diff
@@ -0,0 +1,71 @@
+
+運輸機構 Victor Valley Transit（美國加利福尼亞州維克多維爾）僅對 22 號路線的一部分實施連續停車政策。乘客只能在縣票價區內的任何安全位置上下車。本地票價區內無法連續上車和下車。
+
+Local Fare zone 和 County Fare zone 被 Air Expressway 分開，如下圖所示。這scheduled停止 National Trails Highway - Air Expressway 位於該邊界的稍北處。準確地說，公交機構可以在公交路線與邊界的實際交匯處增加一個停靠站，從那裡可以連續上下車。這個站可能會保留unscheduled.
```
scheduled and unscheduled not translated

josee-sabourin commented on docs/schedule/examples/fares-v1.es.md:
```diff
@@ -0,0 +1,68 @@
+
+<hr/>
+
+Compuesto por [fare_attributes.txt](../../reference/#fare_attributestxt) y [fare_rules.txt](../../reference/#fare_rulestxt), Fares V1 ha sido históricamente el método oficial para describir la información de las tarifas en GTFS. Sin embargo, los dos archivos están limitados en cuanto a la amplitud de los factores que pueden describir eficazmente, y son ambiguos de implementar.[Fares V2](../../exa ...
```
Nit: missing space between period and Fares v2

josee-sabourin commented on docs/schedule/examples/fares-v1.es.md:
```diff
@@ -0,0 +1,68 @@
+- El campo transferencias se deja en blanco para representar las transferencias ilimitadas
+- El campo `transfer_duration` corresponde a la ventana de transferencia de 2 horas (en segundos)
+
+El segundo archivo, [fare_rules.txt](../../reference/#fare_rulestxt) txt asigna las tarifas a los trayectos mediante la vinculación de una tarifa a una ruta así como a un origen/destino en esa ruta.
```
extra txt after hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v1.es.md:
```diff
@@ -0,0 +1,68 @@
+
+## Definir las reglas tarifarias de una agencia
+
+Un trip en la red de metro de la Comisión de Tránsito de Toronto cuesta 3,20 dólares canadienses si los usuarios pagan con la tarjeta PRESTO. Los usuarios también pueden hacer transbordo a otras rutas de metro, tranvía o autobús operadas por la TTC en un plazo de dos horas.
```
unsure if Toronto Transit Commission should have been translated, it wasn't in the German translation

josee-sabourin commented on docs/schedule/examples/fares-v1.es.md:
```diff
@@ -0,0 +1,68 @@
+    Bloor,Bloor Station,,43.670049,-79.385389,ttc_subway_stations
+    Yonge,Yonge Station,,43.671049,-79.386789,ttc_subway_stations
+
+En [fare_rules.txt](../../reference/#fare_rulestxt).txt, la tarifa PRESTO se asocia tanto a las líneas de metro como a las estaciones mediante las siguientes relaciones:
```
extra .txt after hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v1.es.md:
```diff
@@ -0,0 +1,68 @@
+
+En [fare_rules.txt](../../reference/#fare_rulestxt).txt, la tarifa PRESTO se asocia tanto a las líneas de metro como a las estaciones mediante las siguientes relaciones:
+
+- Para `fare_id=presto_fare`, los usuarios pueden viajar entre dos estaciones cualesquiera de la línea 1`(route_id=line1`) y `origin_id=ttc_subway_stations` y `destination_id=ttc_subway_stations`.
```
nit: code format on `route_id=line1` includes starting bracket

josee-sabourin commented on docs/schedule/examples/fares-v1.fr.md:
```diff
@@ -0,0 +1,68 @@
+
+<hr/>
+
+Composé de [fare_attributes.txt](../../reference/#fare_attributestxt) et [fare_rules.txt](../../reference/#fare_rulestxt), Fares V1 a historiquement été la méthode officielle pour décrire les informations tarifaires dans GTFS. Cependant, les deux fichiers sont limités dans l'étendue des facteurs qu'ils peuvent décrire efficacement, et sont ambigus à mettre en œuvre.[Fares V2](../../examples/fares ...
```
nit: space needed between period and Fares V2

josee-sabourin commented on docs/schedule/examples/fares-v1.fr.md:
nit: bracket included in `route_id` code format

josee-sabourin commented on docs/schedule/examples/fares-v1.ja.md:
```diff
@@ -0,0 +1,68 @@
+    Bloor,Bloor Station,,43.670049,-79.385389,ttc_subway_stations
+    Yonge,Yonge Station,,43.671049,-79.386789,ttc_subway_stations
+
+[fare_rules](../../reference/#fare_rulestxt).txtでは、さきがけの運賃は地下鉄の路線と駅に以下のような関係で関連づけられています。
```
nit: .txt not included in hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v1.ja.md:
```diff
@@ -0,0 +1,68 @@
+
+[fare_rules](../../reference/#fare_rulestxt).txtでは、さきがけの運賃は地下鉄の路線と駅に以下のような関係で関連づけられています。
+
+- `fare_id=presto_fareの`場合，ライダーはLine 1の任意の2駅間`（route_id=line1`）を移動でき，`origin_id=ttc_subway_stations`と`destination_id=ttc_subway_stations` である．
```
nit: route_id code format

josee-sabourin commented on docs/schedule/examples/fares-v1.ja.md:
```diff
@@ -0,0 +1,68 @@
+    presto_fare,line1,ttc_subway_stations,ttc_subway_stations
+    presto_fare,line2,ttc_subway_stations,ttc_subway_stations
+
+3番目のファイル[transfers.txtは](../../reference/#transferstxt)、異なるルート間の乗り換えポイントを定義しています。Bloor-Yonge駅での乗り換えをモデル化するには、2つのエントリーが必要です。
```
transfers.txt hyperlink includes Japanese character

josee-sabourin commented on docs/schedule/examples/fares-v1.pt_BR.md:
```diff
@@ -0,0 +1,68 @@
+
+<hr/>
+
+Composto de [fare_attributes.txt](../../reference/#fare_attributestxt) e [fare_rules.txt](../../reference/#fare_rulestxt), Fares V1 tem sido historicamente o método oficial para descrever informações tarifárias no GTFS. Entretanto, os dois arquivos são limitados na amplitude de fatores que eles podem descrever eficientemente, e são ambíguos para implementar. O[Fares V2](../../examples/fares-v2/)  ...
```
nit: space needed between O and Fares V2

josee-sabourin commented on docs/schedule/examples/fares-v1.pt_BR.md:
```diff
@@ -0,0 +1,68 @@
+
+## Definir as regras tarifárias de uma agência
+
+Uma trip na rede de metrô da Comissão de Trânsito de Toronto custa \$3,20 CAD se os cavaleiros pagarem usando o cartão PRESTO. Os cavaleiros também podem transferir para outras linhas de metrô, bonde ou ônibus operados pelo TTC dentro de uma janela de duas horas.
```
Toronto Transit Commission translated

josee-sabourin commented on docs/schedule/examples/fares-v1.pt_BR.md:
```diff
@@ -0,0 +1,68 @@
+
+Em [fare_rules.txt](../../reference/#fare_rulestxt), a tarifa PRESTO é associada tanto às linhas de metrô quanto às estações, usando as seguintes relações:
+
+- Para o `fare_id=presto_fare`, os passageiros podem viajar entre quaisquer duas estações na Linha 1`(route_id=line1`) e `origin_id=ttc_subway_stations` e `destination_id=ttc_subway_stations`.
```
route_id code format

josee-sabourin commented on docs/schedule/examples/fares-v1.ja.md:
```diff
@@ -0,0 +1,68 @@
+
+トロント交通委員会の地下鉄ネットワークの旅は、ライダーがPRESTOカードで支払う場合、\$3.20 CADです。また、2時間以内であれば、TTCが運営する他の地下鉄、路面電車、バス路線に乗り換えることができます。
+
+このサービスは[fare_attributes.txt](../../reference/#fare_attributestxt),[fare_rules.txt](../../reference/#fare_rulestxt),[transfers.txt](../../reference/#transferstxt) というファイルを使って表現することができる．最初のファイル、[fare_attributes.txtは](../../reference/#fare_attributestxt)代理店の運賃を記述し、以下はpresto運賃の例である。
```
spaces needed between commas

josee-sabourin commented on docs/schedule/examples/fares-v1.ru.md:
```diff
@@ -0,0 +1,68 @@
+
+<hr/>
+
+Состоящий из [fare_attributes.txt](../../reference/#fare_attributestxt) и [fare_rules.txt](../../reference/#fare_rulestxt), Fares V1 исторически был официальным методом описания информации о тарифах в GTFS. Однако эти два файла ограничены в широте факторов, которые они могут эффективно описать, и неоднозначны в реализации.[Fares V2](../../examples/fares-v2/) - это проект расширения, находящийся в ...
```
nit: need space between period and Fares V2

josee-sabourin commented on docs/schedule/examples/fares-v1.ru.md:
```diff
@@ -0,0 +1,68 @@
+
+## Определите правила оплаты проезда агентства
+
+trip в метро Торонто стоит \$3,20 CAD, если пассажиры оплачивают проезд картой PRESTO. Всадники также могут пересаживаться на другие маршруты метро, трамваи или автобусы, обслуживаемые TTC, в течение двухчасового окна.
```
TTC translated

josee-sabourin commented on docs/schedule/examples/fares-v1.ru.md:
```diff
@@ -0,0 +1,68 @@
+
+В [fare_rules.txt](../../reference/#fare_rulestxt) тариф PRESTO связан с обеими линиями метро и станциями с помощью следующих отношений:
+
+- При `тарифе fare_id=presto_fare` пассажиры могут перемещаться между любыми двумя станциями линии 1`(route_id=line1`) с `origin_id=ttc_subway_stations` и `destination_id=ttc_subway_stations`.
```
route_id code format

josee-sabourin commented on docs/schedule/examples/fares-v1.zh.md:
```diff
@@ -0,0 +1,68 @@
+- 车费的价格列在价格和`currency_type`下
+- 乘客必须在上车前在车站检票口支付车费。这由`payment_method=1`表示。
+- 转乘字段留空，代表无限制的转乘。
+- `Transfer_duration`字段对应于2小时的换乘窗口（以秒为单位）。
```
transfer has capital T, don't think it should

josee-sabourin commented on docs/schedule/examples/fares-v1.zh.md:
```diff
@@ -0,0 +1,68 @@
+
+在[fare_rules.txt](../../reference/#fare_rulestxt) 中，PRESTO 票价与地铁线路和车站的关系如下。
+
+- 对于`票价_id=presto_fare`，乘客可以在1号线的任何两个车站之间旅行`（routine_id=line1`），并且`origin_id=ttc_subway_stations`和`destination_id=ttc_subway_stations`。
```
fare_id got translated, route_id became routine_id (and code format is off)

josee-sabourin commented on docs/schedule/examples/fares-v1.zh_TW.md:
```diff
@@ -0,0 +1,68 @@
+
+## 定義機構的票價規則
+
+一個trip如果乘客使用 PRESTO 卡付款，在多倫多交通委員會的地鐵網絡上的票價為 3.20 加元。乘客還可以在兩小時內轉乘 TTC 運營的其他地鐵、有軌電車或公交線路。
```
TTC translated

josee-sabourin reviewed (commented):
reviewed 52/143 files, will do the rest tomorrow!

josee-sabourin reviewed (commented):
reviewed 52/143 files, will do the rest tomorrow!

josee-sabourin commented on docs/schedule/examples/fares-v2.de.md:
```diff
@@ -0,0 +1,93 @@
+
+Die obige Datei stellt dies in GTFS mit den folgenden Feldern dar:
+
+- Ein Transfer ist möglich von und zu Strecken, die mit einer PRESTO-Karte bezahlt werden`(ttc_trip_presto`)
```
nit: code format

josee-sabourin commented on docs/schedule/examples/fares-v2.es.md:
```diff
@@ -0,0 +1,93 @@
+
+<hr/>
+
+FaresFares V2 es un proyecto de extensión de GTFS que pretende solucionar las limitaciones de Fares V1. Este proyecto de extensión se está adoptando en iteraciones, y los ejemplos que aparecen a continuación muestran lo que se puede modelar utilizando la parte de Fares Fares V2 que se ha adoptado en la especificación oficial. Consulte más información sobre el proyecto de extensión de Fares Fares  ...
```
Extra Fares at beginning

josee-sabourin commented on docs/schedule/examples/fares-v2.es.md:
```diff
@@ -0,0 +1,93 @@
+
+Algunas agencias de transporte utilizan una estructura tarifaria basada en zonas. Las zonas tarifarias son áreas geográficas divididas asociadas a diferentes precios de tarifas. Tanto si viajan dentro de una misma zona como si lo hacen de una zona a otra, los usuarios deberán conocer la tarifa correcta. En el sistema TTC de Toronto, sólo hay una zona tarifaria. Los usuarios del transporte público ...
+
+Las zonas tarifarias pueden describirse mediante el archivo [stops_areas](../../reference/#stops_areastxt).txt, que asigna las paradas de [stops.txt](../../reference/#stopstxt) a [areas.txt](../../reference/#areastxt).
```
nit: .txt missing from stops_areas.txt

josee-sabourin commented on docs/schedule/examples/fares-v2.es.md:
```diff
@@ -0,0 +1,93 @@
+    area_id,area_name
+    ttc_service_area,TTC Fare Zone
+
+Después, utilizando `stop_id` del archivo [stops.txt](../../reference/#stopstxt).txt, agrupa las paradas a su respectiva área identificada (zona tarifaria).
```
extra .txt after stops.txt

josee-sabourin commented on docs/schedule/examples/fares-v2.es.md:
```diff
@@ -0,0 +1,93 @@
+
+El archivo anterior representa esto en GTFS con los siguientes campos:
+
+- Es posible realizar una transferencia desde y hacia tramos pagados con una tarjeta PRESTO`(ttc_trip_presto`)
```
code text format

josee-sabourin commented on docs/schedule/examples/fares-v2.fr.md:
```diff
@@ -0,0 +1,94 @@
+
+Le fichier ci-dessus représente cela en GTFS avec les champs suivants :
+
+- Un transfert est possible vers et depuis des trajets payés par une carte PRESTO`(ttc_trip_presto`).
```
code text format error

josee-sabourin commented on docs/schedule/examples/fares-v2.id.md:
```diff
@@ -0,0 +1,93 @@
+
+Fares V2 adalahGTFS proyek ekstensi yang bertujuan untuk mengatasi keterbatasanFares V1 . Proyek perluasan ini diadopsi dalam iterasi, dan contoh di bawah menunjukkan apa yang dapat dimodelkan menggunakan porsi dariFares V2 yang telah diadopsi dalam spesifikasi resmi. Lihat informasi lebih lanjut tentangFares V2 proyek ekstensi di bagian [Ekstensi](../../../extensions) .
+
+Untuk sementara, produsen dapat menerapkanFares V2 di samping implementasi dariFares V1 dalam kumpulan data yang sama karena tidak ada konflik teknis antara keduanya. Konsumen akan memiliki pilihan implementasi mana yang akan dikonsumsi secara independen dari yang lain. Dengan adopsi dan dukungan yang memadaiFares V2 ,Fares V1 mungkin akan ditinggalkan di masa mendatang.
```
seems like a lot of spaces missing before any mention of "Fares"

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+
+トロントの交通機関であるTTCを利用するには、運賃を支払う方法がいくつかあります。TTC は、支払い方法によって、運賃の価格を 2 種類用意しています。PRESTO カードで支払うと大人3.20CAD、 現金で支払うと同3 .25CADとなります。
+
+乗り換えのチケットや運賃は、[fare_products.txtという](../../reference/#fare_productstxt)ファイルに記述することができます。各項目は、特定の運賃に対応しています。
```
Japanese characters included in fare_products.txt hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+
+一部の交通機関では、ゾーンベースの運賃体系を採用しています。運賃ゾーンは、異なる運賃価格に関連する地理的な領域を分割したものです。1つのゾーン内を移動する場合でも、あるゾーンから別のゾーンへ移動する場合でも、乗り換え客は正しい運賃を知る必要があります。トロントのTTCでは、運賃ゾーンは1つだけです。トロント市内の移動であれば、どの交通機関を利用しても、同じ運賃を支払えばよいのです。
+
+運賃ゾーンは[stops_areas.txt](../../reference/#stops_areastxt)ファイルを使って記述することができます。[stops.txtから](../../reference/#stopstxt) [areas.txtに](../../reference/#areastxt)停留所を割り当てることで、運賃ゾーンを表現することができます。
```
Japanese characters included in stops.txt and areas.txt hyperlinks

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+    area_id,area_name
+    ttc_service_area,TTC Fare Zone
+
+その後、[stops.txt](../../reference/#stopstxt)ファイルから`stop_idを`使用して、特定されたエリア（運賃ゾーン）ごとに停留所をまとめます。
```
Japanese character included in `stop_id`

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+
+脚グループは、出発地から目的地（またはエリアIDがグループ化された停止に対応している場合は、目的地のセットに起点のセット）までのネットワーク内の旅行を定義します。以下のファイルでは、TTCのネットワーク内のどこにでも移動できる2つのルールを説明しています。最初のルールはさきがけ運賃での移動に対応し、2つ目のルールは現金運賃での移動に対応します。
+
+[**運賃規則(fare_leg_rules.txt)**](../../reference/#fare_leg_rulestxt)
```
Japanese characters in this header

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+
+上のファイルは、これをGTFSで以下のフィールドで表現しています。
+
+- PRESTOカードで決済された脚との間で乗り換えが可能`(ttc_trip_presto`)
```
code text format error

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+上のファイルは、これをGTFSで以下のフィールドで表現しています。
+
+- PRESTOカードで決済された脚との間で乗り換えが可能`(ttc_trip_presto`)
+- `転送回数に制限がないため、transfer_count`は`-1`に設定されています。
```
code text format error

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+
+- PRESTOカードで決済された脚との間で乗り換えが可能`(ttc_trip_presto`)
+- `転送回数に制限がないため、transfer_count`は`-1`に設定されています。
+- `duration_limitは` `7200`秒（2時間相当）に設定されます。
```
Japanese character in `duration_limit`

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+- PRESTOカードで決済された脚との間で乗り換えが可能`(ttc_trip_presto`)
+- `転送回数に制限がないため、transfer_count`は`-1`に設定されています。
+- `duration_limitは` `7200`秒（2時間相当）に設定されます。
+- `duration_limit_typeは` `3`である。これは、運賃支払区間に入ったとき、あるいはバスや路面電車に乗車したときに、さきがけカードをタップしなければならないからである。これは、現在のレグの到着運賃確認と次のレグの到着運賃確認に相当する。
```
Japanese character in `duration_limit_type`

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+- `転送回数に制限がないため、transfer_count`は`-1`に設定されています。
+- `duration_limitは` `7200`秒（2時間相当）に設定されます。
+- `duration_limit_typeは` `3`である。これは、運賃支払区間に入ったとき、あるいはバスや路面電車に乗車したときに、さきがけカードをタップしなければならないからである。これは、現在のレグの到着運賃確認と次のレグの到着運賃確認に相当する。
+- ライダーは最初の運賃のみを支払うので、`fare_transfer_typeは` `0に`設定される。2時間以内の乗り換えには、乗り換え料金や2回目の運賃は発生しない。したがって、コストは初乗り運賃と乗り換え手数料の合計としてモデル化することができる。
```
Japanese character in `fare_transfer_type` and `0`

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+- `duration_limitは` `7200`秒（2時間相当）に設定されます。
+- `duration_limit_typeは` `3`である。これは、運賃支払区間に入ったとき、あるいはバスや路面電車に乗車したときに、さきがけカードをタップしなければならないからである。これは、現在のレグの到着運賃確認と次のレグの到着運賃確認に相当する。
+- ライダーは最初の運賃のみを支払うので、`fare_transfer_typeは` `0に`設定される。2時間以内の乗り換えには、乗り換え料金や2回目の運賃は発生しない。したがって、コストは初乗り運賃と乗り換え手数料の合計としてモデル化することができる。
+- 乗り換え料金は、TTCシステム内では無料であるため、ゼロに設定されている。これは、`fare_product_id=free_transferで`示されます。
```
Japanese character in `fare_product_id=free_transfer`

josee-sabourin commented on docs/schedule/examples/fares-v2.ko.md:
```diff
@@ -0,0 +1,93 @@
+
+<hr/>
+
+Fares V2 이다GTFS 의 한계를 해결하는 것을 목표로 하는 확장 프로젝트Fares V1 . 이 확장 프로젝트는 반복적으로 채택되고 있으며 아래 예제는Fares V2 공식 사양에 채택되었습니다. 더 많은 정보 보기Fares V2 [확장](../../../extensions) 섹션의 확장 프로젝트.
+
+그 동안 생산자는 구현할 수 있습니다.Fares V2 구현과 함께Fares V1 둘 사이에 기술적 충돌이 없기 때문에 동일한 데이터 세트에서. 소비자는 다른 구현과 독립적으로 사용할 구현을 선택할 수 있습니다.Fares V2 ,Fares V1 향후 사용되지 않을 수 있습니다.
```
Spaces needed before mention of "Fares"

josee-sabourin commented on docs/schedule/examples/fares-v2.ko.md:
```diff
@@ -0,0 +1,93 @@
+
+토론토에서 TTC 교통 시스템을 사용하기 위해 요금을 지불하는 방법에는 여러 가지가 있습니다. TTC는 지불 방법에 따라 두 가지 운임 가격 옵션을 제공합니다. PRESTO 카드를 사용하는 경우 성인 요금은 CAD 3.20이며, 현금으로 지불하는 경우 동일한 요금은 CAD 3.25입니다.
+
+대중교통 티켓 또는 운임은 다음을 사용하여 설명할 수 있습니다.[fare_products.txtという](../../reference/#fare_productstxt) 파일. 각 항목은 특정 운임에 해당합니다.
```
inclusion of Korean characters in fare_products.txt hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v2.ko.md:
```diff
@@ -0,0 +1,93 @@
+
+대중교통 티켓 또는 운임은 다음을 사용하여 설명할 수 있습니다.[fare_products.txtという](../../reference/#fare_productstxt) 파일. 각 항목은 특정 운임에 해당합니다.
+
+[fare_products.txtという](../../reference/#fare_productstxt)
```
inclusion of Korean characters in fare_products.txt hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v2.pt_BR.md:
```diff
@@ -0,0 +1,93 @@
+
+O arquivo acima representa isto em GTFS com os seguintes campos:
+
+- É possível uma transferência de e para as pernas que são pagas por um cartão PRESTO`(ttc_trip_presto`)
```
code text format error

josee-sabourin commented on docs/schedule/examples/fares-v2.ru.md:
```diff
@@ -0,0 +1,93 @@
+
+Приведенный выше файл представляет это в GTFS со следующими полями:
+
+- Пересадка возможна на ноги и с ног, оплаченных картой PRESTO`(ttc_trip_presto`).
```
code text format error

josee-sabourin commented on docs/schedule/examples/fares-v2.zh.md:
```diff
@@ -0,0 +1,93 @@
+
+一些公交机构实行基于区域的票价结构。票价区是划分的地理区域，与不同的票价相关。无论他们是在一个区域内旅行还是从一个区域到另一个区域，公交乘客都需要知道正确的票价。在多伦多的TTC系统中，只有一个票价区。公交乘客只需为多伦多市范围内的任何trip距离支付相同的单程票价，无论使用哪种交通形式。
+
+票价区可以用[stops_areas.txt](../../reference/#stops_areastxt)文件来描述，该文件将[stops.txt](../../reference/#stopstxt)txt中的站点分配给[areas.txt](../../reference/#areastxt)。
```
extra txt afer stops.text hyperlink

josee-sabourin commented on docs/schedule/examples/fares-v2.zh.md:
```diff
@@ -0,0 +1,93 @@
+
+上面的文件在GTFS中用以下字段表示。
+
+- 可以在用PRESTO卡支付的路段之间转车`（ttc_trip_presto`）。
```
code text format error

josee-sabourin commented on docs/schedule/examples/fares-v2.zh_TW.md:
```diff
@@ -0,0 +1,93 @@
+    area_id,area_name
+    ttc_service_area,TTC Fare Zone
+
+之後，使用`stop_id`來自[stops.txt](../../reference/#stopstxt) file, group stops 一起到各自的標識區域（fare zone）。
```
"file, group stops" and "(fare zone)" did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.ko.md:
```diff
@@ -0,0 +1,26 @@
+기관 및 서비스에 대한 정보를 제공하는 것 외에도 기관에 대한 정보를 제공하는 것이 가능합니다.GTFS 파일을 사용하는 데이터 세트[feed_info.txt](../../reference/#feed_infotxt) . 여기에는 다음이 포함됩니다.
+
+- 게시자 세부정보
+- 밥을 먹이다language
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.pt_BR.md:
```diff
@@ -0,0 +1,26 @@
+Além de fornecer informações sobre agências e seus serviços, é possível fornecer informações sobre o conjunto de dados GTFS usando o arquivo [feed_info.txt](../../reference/#feed_infotxt). Isto inclui:
+
+- Detalhes da editora
+- language alimentação
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.id.md:
```diff
@@ -0,0 +1,26 @@
+Selain memberikan informasi tentang agensi dan layanannya, Anda juga dapat memberikan informasi tentangGTFS kumpulan data menggunakan file [feed_info.txt](../../reference/#feed_infotxt). Ini termasuk:
+
+- Detail penerbit
+- Memberi makanlanguage
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.fr.md:
```diff
@@ -0,0 +1,26 @@
+En plus de fournir des informations sur les agences et leurs services, il est possible de fournir des informations sur le jeu de données GTFS en utilisant le fichier [feed_info.txt](../../reference/#feed_infotxt). Cela comprend :
+
+- Détails de l'éditeur
+- language flux
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.es.md:
```diff
@@ -0,0 +1,26 @@
+Además de proporcionar información sobre las agencias y sus servicios, es posible proporcionar información sobre el conjunto de datos GTFS utilizando el archivo [feed_info.txt](../../reference/#feed_infotxt). Esto incluye
+
+- Detalles del editor
+- language del feed
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.de.md:
```diff
@@ -0,0 +1,26 @@
+Zusätzlich zu den Informationen über die Agenturen und ihre Dienstleistungen ist es möglich, Informationen über den GTFS in der Datei [feed_info.txt](../../reference/#feed_infotxt) bereitzustellen. Dies beinhaltet:
+
+- Angaben zum Herausgeber
+- language des Feeds
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.ru.md:
```diff
@@ -0,0 +1,26 @@
+Помимо предоставления информации об агентствах и их услугах, можно предоставить информацию о наборе данных GTFS с помощью файла [feed_info.txt](../../reference/#feed_infotxt). Это включает в себя:
+
+- Сведения об издателе
+- language фида
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.zh_TW.md:
```diff
@@ -0,0 +1,26 @@
+除了提供有關機構及其服務的信息外，還可以提供有關GTFS使用文件的數據集[feed_info.txt](../../reference/#feed_infotxt).這包括：
+
+- 出版商詳情
+- 餵養language
```
language did not get translated

josee-sabourin commented on docs/schedule/examples/feed-info.zh_TW.md:
```diff
@@ -0,0 +1,26 @@
+- 飼料有效性
+- 版本
+
+以下是開里奧交通局的一個例子：
```
Transport for Cairo did get translated

josee-sabourin commented on docs/schedule/examples/feed-info.es.md:
```diff
@@ -0,0 +1,26 @@
+- Validez de la fuente
+- Versión
+
+A continuación se muestra un ejemplo de Transporte para Cairio:
```
Transport for Cairo got translated

josee-sabourin commented on docs/schedule/examples/frequencies.es.md:
```diff
@@ -0,0 +1,22 @@
+
+## Describir un servicio basado en la frecuencia
+
+La Société de Transport de Montréal explota los servicios de tránsito de Montreal y gestiona un servicio basado en la frecuencia de sus líneas de metro. Por ello, en lugar de proporcionar un Schedule con las horas de arrival y departure en un conjunto de datos GTFS, se utiliza el archivo frequencies [frequencies.txt](../../reference/#frequenciestxt) txt para describir la frecuencia del servicio a ...
```
extra txt after frequencies.text

josee-sabourin commented on docs/schedule/examples/frequencies.id.md:
```diff
@@ -0,0 +1,22 @@
+
+## Jelaskan layanan berbasis frekuensi
+
+Société de Transport de Montréal mengoperasikan layanan transit di Montréal dan menjalankan layanan berbasis frekuensi untuk jalur métronya. Oleh karena itu, alih-alih memberikan jadwal denganarrival dandeparture kali dalam aGTFS kumpulan data, file[frequencies.txt](../../reference/#frequenciestxt) digunakan untuk menggambarkan frekuensi layanan di seluruh rentang layanan.
```
nit: need space between a and GTFS as well as file and frequencies.txt

josee-sabourin commented on docs/schedule/examples/frequencies.ja.md:
```diff
@@ -0,0 +1,23 @@
+
+## 頻度ベースのサービスを記述する
+
+モントリオール交通局（Société de Transport de Montréal）は、モントリオールの交通サービスを運営しており、そのメトロー路線では頻度ベースのサービスを実施しています。そのため，GTFSデータセットでは発着時刻を記載したスケジュールを提供する代わりに，[frequencies.txtという](../../reference/#frequenciestxt)ファイルを用いて，運行期間中のサービス頻度を記載している．
```
Japanese characters in frequencies.txt hyperlink

josee-sabourin commented on docs/schedule/examples/frequencies.zh.md:
```diff
@@ -0,0 +1,22 @@
+    22M-GLOBAUX-00-S_1_2,16:01:25,16:19:25,180
+    22M-GLOBAUX-00-S_1_2,16:19:25,17:03:25,165
+
+上面，一个在绿线上的trip，`行程号=22M-GLOBAUX-00-S_1_2`被用作一个例子。对于这个trip，第一条记录表明在下午4:01:25到4:19:25之间，每180秒（或3分钟）有一趟列车。同样，第二条记录显示，在下午4:19:25到5:03:25之间，每隔165秒就有一趟列车运行。
```
trip did not get translated twice

josee-sabourin commented on docs/schedule/examples/frequencies.zh_TW.md:
```diff
@@ -0,0 +1,22 @@
+
+## 描述基於頻率的服務
+
+Société de Transport de Montréal 在蒙特利爾經營交通服務，並為其地鐵線路提供基於頻率的服務。因此，與其提供時間表，不如arrival和departure次在GTFS數據集，文件[frequencies.txt](../../reference/#frequenciestxt)用於描述整個服務跨度內的服務頻率。
```
arrival, departure did not get translated

josee-sabourin commented on docs/schedule/examples/frequencies.zh_TW.md:
```diff
@@ -0,0 +1,22 @@
+    22M-GLOBAUX-00-S_1_2,16:01:25,16:19:25,180
+    22M-GLOBAUX-00-S_1_2,16:19:25,17:03:25,165
+
+以上，一個trip以綠線上的`trip_id=22M-GLOBAUX-00-S_1_2`為例。為了那個原因trip，第一個記錄表示在下午 4:01:25 到下午 4:19:25 之間，火車每 180 秒（或 3 分鐘）運行一次。同樣，第二條記錄表明在下午 4:19:25 到下午 5:03:25 之間，火車每 165 秒運行一次。
```
trip did not get translated twice

josee-sabourin commented on docs/schedule/examples/index.de.md:
```diff
@@ -0,0 +1,21 @@
+  - [Fahrwege](pathways)
+  - [Übersetzungen](translations)
+  - [Informationen zur Einspeisung](feed-info)
+  - [Datensatz-Attribute](attributions)
```
I'll only make this comment once but applies to all languages for this page - I am unsure which of these links should be translated or not. (ex: Fares v1 and v2 getting translated here)

josee-sabourin commented on docs/schedule/examples/fares-v2.ja.md:
```diff
@@ -0,0 +1,93 @@
+  exclude: true
+---
+
+# 運賃v2
```
Fares got translated, it isn't translated in other languages

josee-sabourin commented on docs/schedule/examples/index.ja.md:
```diff
@@ -0,0 +1,21 @@
+  - [翻訳](translations)
+  - [フィード情報](feed-info)
+  - [データセットの帰属](attributions)
+- 実際の交通機関が公開しているフィードについては、[OpenMobilityDataを](https://openmobilitydata.org/)参照してください。
```
Japanese character in OpenMobilityData hyperlink

josee-sabourin commented on docs/schedule/examples/index.ko.md:
```diff
@@ -0,0 +1,21 @@
+  exclude: true
+---
+
+# GTFS Schedule 예
```
Schedule did not get translated

josee-sabourin commented on docs/schedule/examples/index.pt_BR.md:
```diff
@@ -0,0 +1,21 @@
+  exclude: true
+---
+
+# Exemplos deGTFS Schedule
```
Schedule did not get translated

josee-sabourin commented on docs/schedule/examples/index.ru.md:
```diff
@@ -0,0 +1,21 @@
+  exclude: true
+---
+
+# ПримерыGTFS Schedule
```
Schedule did not get translated

josee-sabourin commented on docs/schedule/examples/index.ru.md:
```diff
@@ -0,0 +1,21 @@
+
+Чтобы помочь в понимании спецификации GTFS и в создании инструментов, которые читают и записывают данные GTFS, посмотрите на следующие примеры фидов GTFS:
+
+- На [странице "Пример GTFS Feed](/schedule/example-feed) " представлен простой GTFS Feed, который можно использовать в качестве модели для понимания того, как составляется GTFS Feed.
```
Feed did not get translated

josee-sabourin commented on docs/schedule/examples/index.zh.md:
```diff
@@ -0,0 +1,21 @@
+  exclude: true
+---
+
+# GTFS Schedule实例
```
Schedule did not get translated

josee-sabourin commented on docs/schedule/examples/index.zh_TW.md:
```diff
@@ -0,0 +1,21 @@
+  exclude: true
+---
+
+# GTFS Schedule例子
```
Schedule did not get translated

josee-sabourin commented on docs/schedule/examples/index.zh_TW.md:
```diff
@@ -0,0 +1,21 @@
+
+幫助理解GTFS規範和生產讀寫工具GTFS數據，看看下面的例子GTFS提要：
+
+- [例子提要頁面](/schedule/example-feed)提供了一個簡單的GTFS Feed 可以用作模型來了解如何GTFS飼料放在一起。
```
Feed did not get translated

josee-sabourin commented on docs/schedule/examples/pathways.de.md:
```diff
@@ -0,0 +1,79 @@
+
+## Beschreibung der Treppen und Rolltreppen
+
+Der Eingang der Waterfront Station in der Granville Street verfügt über einen Aufzug, eine Rolltreppe und eine Treppe; die Eingänge sind als Knoten oben in [stops.txt](../../reference/#stopstxt) definiert. Um die Eingänge mit den inneren Bereichen des Bahnhofs zu verbinden, müssen zusätzliche Knoten in [stops.txt](../../reference/#stopstxt) unter der `parent_station` von Waterfront Station erstel ...
```
nit: code text format error for `location_type 3`

josee-sabourin commented on docs/schedule/examples/pathways.de.md:
```diff
@@ -0,0 +1,79 @@
+    95,Waterfront Station Granville Stair Landing, 49.285169,-123.114198,3,12034,
+    96,Waterfront Station Granville Escalator Landing,49.285183,-123.114222,3,12034,
+
+<img class="center" src="../../../assets/pathways.png" width="700px"/>
```
all three images 404 for all languages, wonder if this is a problem on my end or not, flagging because they rendered fine on the continuous stops page

josee-sabourin commented on docs/schedule/examples/pathways.de.md:
```diff
@@ -0,0 +1,79 @@
+
+## Beschreiben Sie Aufzüge und Fahrwege
+
+Der Aufzug in der Granville Street bringt die Fahrgäste zu einem Weg auf der Ebene der Bahnhofshalle, wo die Rolltreppe und die Treppe end. Der Aufzug auf der oberirdischen Ebene ist bereits als Bahnhofseingang oben definiert`stop_id` `92`). Daher muss auch die Aufzugstür in der Bahnhofshalle definiert werden.
```
nit: missing bracket before stop_id 92

josee-sabourin commented on docs/schedule/examples/pathways.es.md:
```diff
@@ -0,0 +1,79 @@
+
+Con GTFS, es posible describir con precisión las estaciones utilizando información sobre las entradas y el interior de las mismas. Este ejemplo describe secciones de la estación de Waterfront en el centro de Vancouver. La estación forma parte de la red de Skytrain de la ciudad y a ella llegan la Canada Line, la Expo Line, el SeaBus y el West Coast Express. Hay tres entradas a nivel de calle que p ...
+
+En primer lugar, la ubicación de la estación y sus entradas se definen en stops [stops.txt](../../reference/#pathwaystxt)txt:
```
extra txt after hyperlink

josee-sabourin commented on docs/schedule/examples/pathways.es.md:
```diff
@@ -0,0 +1,79 @@
+
+## Describir las escaleras y escaleras mecánicas
+
+La entrada de la estación Waterfront en la calle Granville tiene un ascensor, una escalera mecánica y escaleras, las entradas se definen como nodos arriba en stops. [stops.txt](../../reference/#stopstxt). Para conectar las entradas a las secciones interiores de la estación, hay que crear nodos adicionales en stops [stops.txt](../../reference/#stopstxt) txt bajo el `parent_station` de Waterfront S ...
```
extra txt after 2nd stops.txt hyperlink
code format error for `location_type 3`

josee-sabourin commented on docs/schedule/examples/pathways.es.md:
```diff
@@ -0,0 +1,79 @@
+
+<img class="center" src="../../../assets/pathways.png" width="700px"/>
+
+A continuación, se utiliza el archivo [pathways.txt](../../reference/#pathwaystxt).txt para enlazar los nodos y crear caminos, donde el primer registro enlaza los nodos correspondientes a la parte superior e inferior de las escaleras. El `pathway_mode` se establece en `2` para indicar las escaleras, y el último campo describe que los pasajeros pueden ir en ambos sentidos (arriba y abajo) en las e ...
```
extra txt after pathways.txt

josee-sabourin commented on docs/schedule/examples/pathways.es.md:
```diff
@@ -0,0 +1,79 @@
+
+A continuación, se utiliza el archivo [pathways.txt](../../reference/#pathwaystxt).txt para enlazar los nodos y crear caminos, donde el primer registro enlaza los nodos correspondientes a la parte superior e inferior de las escaleras. El `pathway_mode` se establece en `2` para indicar las escaleras, y el último campo describe que los pasajeros pueden ir en ambos sentidos (arriba y abajo) en las e ...
+
+Del mismo modo, el segundo registro describe las escaleras mecánicas`(pathway_mode` fijado en `4`). Dado que las escaleras mecánicas sólo pueden moverse en una dirección, el campo `is_bidirectional` se establece en `0`, por lo que la escalera mecánica se mueve en un solo sentido, del nodo `96` al `91` (hacia arriba).
```
code format error for `pathway mode`

josee-sabourin commented on docs/schedule/examples/pathways.es.md:
```diff
@@ -0,0 +1,79 @@
+
+<img class="center" src="../../../assets/pathways-3.png" width="500px"/>
+
+Por último, los nodos se conectan entre sí para definir el trayecto subterráneo, como se muestra en el archivo [pathways.txt](../../reference/#pathwaystxt) txt que aparece a continuación:
```
extra txt after pathways.txt

josee-sabourin commented on docs/schedule/examples/pathways.fr.md:
```diff
@@ -0,0 +1,79 @@
+
+## Décrire les escaliers et les escalators
+
+L'entrée de la station Waterfront à Granville street a un ascenseur, un escalator et des escaliers, les entrées sont définies comme des noeuds ci-dessus dans [stops.txt](../../reference/#stopstxt). Pour connecter les entrées aux sections intérieures de la station, des noeuds supplémentaires doivent être créés dans le fichier [stops.txt](../../reference/#stopstxt) sous le `parent_station` de Water ...
```
code format error for `location_type 3`

josee-sabourin commented on docs/schedule/examples/pathways.fr.md:
```diff
@@ -0,0 +1,79 @@
+
+Ensuite, le fichier [pathways.txt](../../reference/#pathwaystxt) est utilisé pour relier les nœuds afin de créer des chemins, où le premier enregistrement relie les nœuds correspondant au haut et au bas de l'escalier. Le `pathway_mode` est défini sur `2` pour indiquer les escaliers, et le dernier champ décrit que les passagers peuvent aller dans les deux sens (vers le haut et vers le bas) sur les ...
+
+De même, le deuxième enregistrement décrit l'escalier mécanique`(pathway_mode` défini sur `4`). Comme les escaliers mécaniques ne peuvent se déplacer que dans une seule direction, le champ `is_bidirectional` est défini à `0`, et l'escalier mécanique se déplace donc dans un seul sens, du nœud `96` au `91` (vers le haut).
```
code format error for `pathway_mode`

josee-sabourin commented on docs/schedule/examples/pathways.id.md:
```diff
@@ -0,0 +1,79 @@
+
+## Jelaskan lift dan jalur
+
+Lift di jalan Granville membawa penumpang ke jalur di tingkat concourse tempat eskalator dan tangga beradaend . Lift di tingkat permukaan sudah didefinisikan sebagai pintu masuk stasiun di atas (stop_id `92` ). Oleh karena itu, pintu lift di tingkat concourse juga perlu ditentukan.
```
stop_id not formatted as code text

josee-sabourin commented on docs/schedule/examples/pathways.ja.md:
```diff
@@ -0,0 +1,79 @@
+
+GTFSでは、出入口や駅構内の情報を用いて、駅を正確に表現することができます。この例では、バンクーバーのダウンタウンにあるウォーターフロント駅の一部を記述しています。この駅はスカイトレインの一部であり、カナダライン、エキスポライン、シーバス、ウエストコーストエクスプレスが乗り入れている。駅には3つの路面入口があり、乗客はここから出入りします。駅の残りの部分は地下にあり、料金の確認ができるコンコースレベルと、ホームのある低層部があります。
+
+まず、駅の位置と入り口を[stops.txtで](../../reference/#pathwaystxt)定義する。
```
Japanese character in stops.txt hyperlink

josee-sabourin commented on docs/schedule/examples/pathways.ja.md:
```diff
@@ -0,0 +1,79 @@
+    93,Waterfront Station Entrance on Cordova,49.285607,-123.111993,2,12034,1
+    94,Waterfront Station Entrance on Howe,49.286898,-123.113367,2,12034,2
+
+上のファイルでは、最初のレコードが駅の位置に関するもので、したがって`location_type`は`1` に設定されている。他の 5 つのレコードは 3 つの駅の入口に関するものである（グランビル入口は実際には階段、エスカレーター、エレベーターの 3 つの別々の入口を持っているので 5 つのレコードが必要である）。この5つのレコードは、`location_type`が`3 に`設定されているので、エントランスと定義される。
```
Japanese character formatted as code after `3`

josee-sabourin commented on docs/schedule/examples/pathways.ja.md:
```diff
@@ -0,0 +1,79 @@
+
+上のファイルでは、最初のレコードが駅の位置に関するもので、したがって`location_type`は`1` に設定されている。他の 5 つのレコードは 3 つの駅の入口に関するものである（グランビル入口は実際には階段、エスカレーター、エレベーターの 3 つの別々の入口を持っているので 5 つのレコードが必要である）。この5つのレコードは、`location_type`が`3 に`設定されているので、エントランスと定義される。
+
+また、エントランスの`parent_station`には、Waterfront Station の`stop_id`が記載されており、駅と関連付けられています。アクセシブルな入口は`wheelchair_boardingを` `1`、ノンアクセシブルな入口は`2を`設定しています。
```
Japanese character formatted as code after `wheel_chair boarding` and `2`

josee-sabourin commented on docs/schedule/examples/pathways.ja.md:
```diff
@@ -0,0 +1,79 @@
+
+## 階段とエスカレーターの記述
+
+Granville 通りにある Waterfront 駅の入り口には、エレベーター、エスカレーター、階段があり、入り口は[stops.txt](../../reference/#stopstxt) の上のノードとして定義されています。エントランスと駅の内部をつなぐには、[stops.txt](../../reference/#stopstxt)の`parent_station`の Waterfront Station の下にノードを追加する必要があります。下の[stops.txt](../../reference/#stopstxt)には、階段とエスカレーターの下に相当する汎用ノード`(location_type 3`) が定義されています。
```
code text format error for `location_type 3`

josee-sabourin commented on docs/schedule/examples/pathways.ja.md:
```diff
@@ -0,0 +1,79 @@
+
+次に、[pathways.txt](../../reference/#pathwaystxt)ファイルを使用してノードをリンクして経路を作成し、最初のレコードで階段の上部と下部に関連するノードをリンクしています。`pathway_mode`は階段を示す`2`に設定され、最後のフィールドには乗客が階段の上下を行き来できることが記述されている。
+
+同様に、2番目のレコードでは、エスカレーターを記述します`（pathway_modeは` `4に`設定）。エスカレーターは一方向にしか移動できないので、`is_bidirectional`フィールドは`0に`設定されており、ノード`96から` `91`（上方向）へ一方通行で移動するようになっています。
```
Japanese character formatted as code in `pathway_mode`, `4`, `0`, and `96` as well as bracket included in code text for `pathway_mode`

josee-sabourin commented on docs/schedule/examples/pathways.ja.md:
```diff
@@ -0,0 +1,79 @@
+
+## エレベータと通路の記述
+
+グランビルストリートにあるエレベータは、エスカレータと階段の終点であるコンコースレベルにある通路に乗客を運びます。地上階のエレベータは、すでに上の駅入口として定義されている`（stop_id` `92`）。従って、コンコースレベルのエレベータドアも定義する必要がある。
```
code text format error

josee-sabourin commented on docs/schedule/examples/pathways.ko.md:
```diff
@@ -0,0 +1,79 @@
+
+와 함께GTFS , 입구 및 역 내부 정보를 이용하여 정확한 역 설명이 가능합니다. 이 예는 밴쿠버 시내에 있는 워터프론트 역의 섹션을 설명합니다. 이 역은 도시 스카이트레인 네트워크의 일부이며 Canada Line, Expo Line, SeaBus 및 West Coast Express가 운행합니다. 3개의 거리 수준 입구를 통해 라이더는 역에 출입할 수 있습니다. 역의 나머지 부분은 요금 확인을 위한 중앙 홀 수준과 플랫폼이 있는 낮은 수준이 있는 지하에 있습니다.
+
+첫째, 역의 위치와 입구는 다음과 같이 정의됩니다.[stops.txtで](../../reference/#pathwaystxt) :
```
Korean character included in stops.txt hyperlink

josee-sabourin commented on docs/schedule/examples/pathways.ko.md:
```diff
@@ -0,0 +1,79 @@
+
+## 계단과 에스컬레이터 설명
+
+Granville Street의 Waterfront Station 입구에는 엘리베이터, 에스컬레이터 및 계단이 있으며 입구는 위의 노드로 정의됩니다.stops.txt . 역사 내부 구간의 입구를 연결하기 위해서는 추가 노드를 생성해야 합니다.stops.txt 워터프론트 스테이션의 `parent_station` 아래. 에서stops.txt 아래 파일에는 계단과 에스컬레이터의 하단에 해당하는 일반 노드( `location_type 3` )가 정의되어 있습니다.
```
all stops.txt not hyperlinked

josee-sabourin commented on docs/schedule/examples/pathways.ko.md:
```diff
@@ -0,0 +1,79 @@
+
+<img class="center" src="../../../assets/pathways.png" width="700px"/>
+
+다음으로 파일pathways.txt 노드를 연결하여 통로를 만드는 데 사용되며, 여기서 첫 번째 레코드는 계단의 상단과 하단에 속하는 노드를 연결합니다. `pathway_mode` 는 계단을 나타내기 위해 `2` 로 설정되고 마지막 필드는 승객이 계단에서 양방향(위와 아래)으로 이동할 수 있음을 설명합니다.
```
pathways.txt not hyperlinked

josee-sabourin commented on docs/schedule/examples/pathways.ko.md:
```diff
@@ -0,0 +1,79 @@
+
+## 엘리베이터 및 통로 설명
+
+Granville 거리의 엘리베이터는 승객을 에스컬레이터와 계단이 있는 중앙 홀 수준의 통로로 안내합니다.end . 지표면의 엘리베이터는 위의 역 입구로 이미 정의되어 있습니다(stop_id `92` ). 따라서 콩코스 레벨의 엘리베이터 도어도 정의해야 합니다.
```
stop_id not formatted as code

josee-sabourin commented on docs/schedule/examples/pathways.pt_BR.md:
```diff
@@ -0,0 +1,79 @@
+
+## Descreva as escadas e escadas rolantes
+
+A entrada da estação Waterfront na rua Granville tem um elevador, uma escada rolante e escadas, as entradas são definidas como nós acima em [stops.txt](../../reference/#stopstxt). Para conectar as entradas às seções internas da estação, nós adicionais têm de ser criados em [stops.txt](../../reference/#stopstxt) sob a `parent_station` estação Waterfront. No arquivo [stops.txt](../../reference/#sto ...
```
`location_type` format error

josee-sabourin commented on docs/schedule/examples/pathways.pt_BR.md:
```diff
@@ -0,0 +1,79 @@
+
+Em seguida, o arquivo [pathways.txt](../../reference/#pathwaystxt) é usado para ligar os nós para criar caminhos, onde o primeiro registro liga os nós pertencentes à parte superior e inferior das escadas. O `pathway_mode` é definido como `2` para indicar escadas, e o último campo descreve que os passageiros podem ir para ambos os lados (para cima e para baixo) das escadas.
+
+Da mesma forma, o segundo registro descreve a escada rolante`(pathway_mode` definido como `4`). Como as escadas rolantes só podem se mover em uma direção, o campo `is_bidirecional` é ajustado para `0`, portanto a escada rolante se move em uma direção, do nó `96` para `91` (para cima).
```
`pathway_mode` format error

josee-sabourin commented on docs/schedule/examples/pathways.ru.md:
```diff
@@ -0,0 +1,79 @@
+
+## Опишите лестницы и эскалаторы
+
+Вход в Waterfront Station на Granville street имеет лифт, эскалатор и лестницу, входы определены как узлы выше в [stops.txt](../../reference/#stopstxt). Чтобы соединить входы с внутренними секциями станции, необходимо создать дополнительные узлы в файле [stops.txt](../../reference/#stopstxt) под `родительской_станцией` Waterfront Station. В файле [stops.txt](../../reference/#stopstxt) ниже опреде ...
```
parent_station got translated
location_type 3 format error

josee-sabourin commented on docs/schedule/examples/pathways.ru.md:
```diff
@@ -0,0 +1,79 @@
+
+Далее, файл [pathways.txt](../../reference/#pathwaystxt) используется для связи узлов для создания путей, где первая запись связывает узлы, относящиеся к верху и низу лестницы. `pathway_mode` установлен на `2`, чтобы указать лестницу, а последнее поле описывает, что пассажиры могут идти по лестнице в обе стороны (вверх и вниз).
+
+Аналогично, во второй записи описывается эскалатор`(pathway_mode` установлен на `4`). Поскольку эскалаторы могут двигаться только в одном направлении, поле `is_bidirectional` установлено в `0`, следовательно, эскалатор движется в одну сторону, от узла `96` к `91` (вверх).
```
pathway_mode format error

josee-sabourin commented on docs/schedule/examples/pathways.zh.md:
```diff
@@ -0,0 +1,79 @@
+
+## 描述车站入口和出口的位置
+
+通过GTFS，可以使用入口和车站内部的信息来准确地描述车站。这个例子描述了温哥华市中心的Waterfront站的部分。该站是该市天铁网络的一部分，有加拿大线、世博线、海上巴士和西海岸快线。三个街道层面的入口允许乘客进入和离开车站。车站的其余部分在地下，有一个用于验票的大厅层和一个带有月台的低层。
```
Canada Line, Expo Line, SeaBus, and West Coast Express got translated

josee-sabourin commented on docs/schedule/examples/pathways.zh.md:
```diff
@@ -0,0 +1,79 @@
+
+在上述文件中，第一条记录与车站位置有关，因此，`location_type`被设置为`1`。 其他五条记录与车站的三个入口有关（需要五条记录，因为格兰维尔入口实际上有三个独立的入口，一个楼梯，一个自动扶梯和一个电梯）。这五条记录被定义为出入口，因为`location_type`被设置为`3`。
+
+此外，Waterfront站的`stop_id`被列在出入口的`父站`下，以便将它们与车站联系起来。无障碍出入口的`轮椅`设置为`1`，非无障碍出入口的轮椅设置为`2`。
```
parent_station and wheelchair_boarding got translated

josee-sabourin commented on docs/schedule/examples/pathways.zh.md:
```diff
@@ -0,0 +1,79 @@
+
+## 描述楼梯和自动扶梯
+
+位于Granville街的Waterfront车站的入口有一个电梯、一个自动扶梯和楼梯，这些入口在[stops.txt](../../reference/#stopstxt)中被定义为上述节点。为了将入口连接到车站的内部区域，必须在Waterfront站的`父站`下的[stops.txt](../../reference/#stopstxt)中创建额外的节点。在下面的[stops.txt](../../reference/#stopstxt)文件中，定义了对应于楼梯和自动扶梯底部的通用节点`（location_type 3`）。
```
parent_station translated
location_type 3 format error

josee-sabourin commented on docs/schedule/examples/pathways.zh.md:
```diff
@@ -0,0 +1,79 @@
+
+接下来，文件[pathways.txt](../../reference/#pathwaystxt)被用来连接节点以创建路径，其中第一条记录连接了与楼梯顶部和底部有关的节点。`pathway_mode`被设置为`2`，表示楼梯，最后一个字段描述了乘客可以在楼梯上双向（向上和向下）行走。
+
+同样地，第二条记录描述了自动扶梯`（pathway_mode`设置为`4`）。由于自动扶梯只能向一个方向移动，字段`is_bidirectional`被设置为`0`，因此自动扶梯单向移动，从节点`96`到`91`（向上）。
```
pathway_mode format error

josee-sabourin commented on docs/schedule/examples/pathways.zh.md:
```diff
@@ -0,0 +1,79 @@
+
+## 描述电梯和路径
+
+格兰维尔街的电梯将乘客带到大厅层的一个通道上，那里是自动扶梯和楼梯end。地面层的电梯已经被定义为上面的一个车站入口`stop_id` `92`）。因此，大厅层的电梯门也需要被定义。
```
opening bracket vanished

josee-sabourin commented on docs/schedule/examples/pathways.zh_TW.md:
```diff
@@ -0,0 +1,79 @@
+
+## 描述車站出入口的位置
+
+和GTFS ，可以使用有關入口和車站內部的信息準確描述車站。此示例描述了溫哥華市中心的 Waterfront 車站部分。該站是該市輕軌網絡的一部分，由加拿大線、世博線、海上巴士和西海岸快車提供服務。三個街道入口允許騎手進出車站。車站的其餘部分位於地下，設有一個用於驗證票價的大廳層和一個設有站台的較低層。
```
Canada Line, Expo Line, SeaBus, and West Coast Express got translated

josee-sabourin reviewed (commented):
100 out 143 done, will come back later today to finish the rest!

josee-sabourin reviewed (commented):
100 out 143 done, will come back later today to finish the rest!

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+Das fünfte Feld`route_type`) wird verwendet, um die verschiedenen Arten von Linien zu unterscheiden:
```
vanishing bracket

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+
+- `stop_id` ist ein eindeutiger Bezeichner
+- `stop_code` und `stop_name` enthalten in der Regel fahrerbezogene Informationen
+- Der genaue Standort wird anhand von Koordinaten`(stop_lat` und `stop_lon`) angegeben.
```
code text format error for stop_lat

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_id` ist ein eindeutiger Bezeichner
+- `stop_code` und `stop_name` enthalten in der Regel fahrerbezogene Informationen
+- Der genaue Standort wird anhand von Koordinaten`(stop_lat` und `stop_lon`) angegeben.
+- Das sechste Feld`(location_type`) dient der Unterscheidung zwischen Haltestellen und Bahnhöfen
```
code text format error

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+
+Hier wird ein Dienst beschrieben, der nur samstags und sonntags verkehrt. Daher werden die Felder für diese Tage mit 1 und die Felder für die übrigen Tage mit Null gefüllt. Dieser Dienst läuft vom 23. Juni 2022 bis zum 3. September 2022, wie in den Feldern `start_date` und `end_date` angegeben.
+
+In diesem Beispiel beschreibt die Datei [trips.txt](../../reference/#tripstxt) xt 3 Wochenendfahrten, die von der oben beschriebenen MAX Orange Route bedient werden.
```
extra xt after trips.txt hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+- Das sechste Feld`(location_type`) dient der Unterscheidung zwischen Haltestellen und Bahnhöfen
+- Der erste Datensatz entspricht einer Bushaltestelle, also `location_type=0`
+- Der zweite Datensatz entspricht einer Haltestelle, also `location_type=1`
+- Eine vollständige Liste der Werte für ` location_type  `finden Sie [hier](../../reference/stopstxt)
```
extra space in code text

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+- Die `service_id` aus [calendar.txt](../../reference/#calendartxt), die den Wochenenden entspricht, wird aufgelistet
+- Jeder Datensatz enthält eine eindeutige id für jede trip
+
+Der text des Fahrtziels wird angegeben, der normalerweise auf Schildern innerhalb und außerhalb des Busses angezeigt wird.
```
errant line break

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+    weekend_service,20220623,2
+
+- Die `service_id` `weekend_service` wird aufgelistet
+- Das Datum des gestrichenen oder zusätzlicher Dienst ist unter `Datum` aufgeführt (17. Juli 2022)
```
`date` got translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone
+    CT,Calgary Transit,http://www.calgarytransit.com,America/Edmonton,,403-262-1000
+
+Calgary Transit opera LRT, BRT, servicio regular de autobús, paratránsito y tránsito a demanda en Calgary, AB. En este ejemplo, se definen dos rutas, la primera es un autobús y la segunda es un LRT. Utilizando el archivo [routes.txt](../../reference/#routestxt)txt, se asigna a cada ruta un id único y un nombre corto y otro largo para facilitar la lectura.
```
extra txt after routes.txt hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+El quinto campo`route_type`) se utiliza para diferenciar los tipos de rutas:
```
missing opening bracket

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+
+## Paradas
+
+En GTFS, las paradas y las estaciones se describen utilizando el archivo [stops.txt](../../reference/#stopstxt)txt, a continuación, una parada de autobús se define en el primer registro y una estación de LRT se define en el segundo registro.
```
extra txt after stops.txt hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_id` es un identificador único
+- `stop_code` y `stop_name` suelen contener información orientada al usuario
+- La ubicación exacta se proporciona mediante coordenadas`(stop_lat` y `stop_lon`)
+- El sexto campo`(location_type`) se utiliza para diferenciar las paradas de las estaciones
```
code formatted opened brackets in `stop_lat` and `location_type`

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+- El sexto campo`(location_type`) se utiliza para diferenciar las paradas de las estaciones
+- El primer registro corresponde a una parada de autobús, por lo que `location_type=0`
+- El segundo registro corresponde a una estación, por lo que `location_type=1`
+- La lista completa de valores para ` location_type  `se encuentra [aquí](../../reference/stopstxt)
```
extra code formatted space after location_type

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+
+Una vez descritas las rutas de la agencia, es posible describir los viajes a los que da servicio cada ruta.
+
+En primer lugar, es necesario definir el intervalo de servicio mediante calendar [calendar.txt](../../reference/#calendartxt)txt.
```
extra txt

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+
+En este caso, se describe un servicio que sólo funciona los sábados y domingos, por lo que los campos de esos días se rellenan con 1, y los del resto de días se rellenan con cero. Este servicio funciona desde el 23 de junio de 2022 hasta el 3 de septiembre de 2022, como se muestra en los campos `start_date` y `end_date`.
+
+En este ejemplo, el archivo [trips.txt](../../reference/#tripstxt).txt describe 3 viajes de fin de semana que son servidos por la ruta MAX Naranja que fue descrita anteriormente.
```
extra .txt

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+- El `service_id` de calendar. [calendar.txt](../../reference/#calendartxt) que corresponde a los fines de semana está listado
+- Cada registro contiene un id único para cada trip
+
+Se proporciona el text de la señal de cabeza, que es lo que suele aparecer en las señales dentro y fuera del autobús
```
errant line break

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+
+Es posible definir excepciones al servicio como los días de servicio agregado (días especiales) o los días de servicio eliminados (como el no servicio en días festivos).
+
+Por ejemplo, si no hay servicio programado el domingo 17 de julio de 2022 - entonces esa fecha se puede eliminar de `weekend_service` en [calendar.txt](../../reference/#calendartxt) txt dividiendo el servicio en dos:
```
extra txt

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+| `weekend_service1` | `20220623` | `20220716` |
+| `weekend_service2` | `20220718` | `20220903` |
+
+Sin embargo, esto complica el archivo, ya que `service_id` se rompe en dos y esta ruptura se trasladará en cascada a [trips.txt](../../reference/#tripstxt).txt. En su lugar, esto se puede hacer de una manera más fácil utilizando [calendar_dates.txt](../../reference/#calendar_datestxt).txt como se muestra a continuación:
```
extra txt after both hyperlinks

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+    weekend_service,20220623,2
+
+- El `service_id` `weekend_service` aparece en la lista
+- La fecha del servicio eliminado o adicional aparece en la `fecha` (17 de julio de 2022)
```
`date` got translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.fr.md:
missing opening bracket

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.fr.md:
code formatted opening brackets

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.fr.md:
code formatted space

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.fr.md:
errant line break

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.id.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+Bidang kelima (route_type ) digunakan untuk membedakan jenis rute:
```
route_type not formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.id.md:
```diff
@@ -0,0 +1,134 @@
+- Itu `service_id` dari [calendar.txt](../../reference/#calendartxt) yang sesuai dengan akhir pekan terdaftar
+- Setiap catatan berisi unikid untuk setiaptrip
+
+Tanda kepalatext disediakan, yang biasanya ditampilkan pada tanda di dalam dan di luar bus
```
errant line break

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone
+    CT,Calgary Transit,http://www.calgarytransit.com,America/Edmonton,,403-262-1000
+
+Calgary Transitは、AB州カルガリーでLRT、BRT、通常のバスサービス、パラトランジット、オンデマンド輸送を運営しています。この例では、2つのルートが定義されています。1つ目はバスで、2つ目はLRTです。[routes.txtという](../../reference/#routestxt)ファイルを使って、各ルートに一意のID、短い名前、そして人間が読みやすいように長い名前を割り当てています。
```
Japanese characters included in hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+5番目のフィールド`（route_type`）は、路線の種類を区別するために使用される。
```
code text format error

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+5番目のフィールド`（route_type`）は、路線の種類を区別するために使用される。
+
+- 1番目はバスなので、`route_type=3`
+- 2つ目はLRTであるため、rout`e_type=0と`なる。
```
code text format error

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+
+- 1番目はバスなので、`route_type=3`
+- 2つ目はLRTであるため、rout`e_type=0と`なる。
+- `route_typeの`値の完全なリストは、[routes.txt](../../reference/#routestxt)あります。
```
Japanese character included in `route_type` code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+    6810,6810,NB Marlborough CTrain Station,51.058990,-113.981582,0
+
+- `stop_idは`一意な識別子である．
+- `stop_codeと` `stop_nameは`、通常、ライダー向けの情報を含みます。
```
Japanese characters in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_idは`一意な識別子である．
+- `stop_codeと` `stop_nameは`、通常、ライダー向けの情報を含みます。
+- 正確な位置は、座標`(stop_lat`と`stop_lon`) を使用して提供されます。
+- 6番目のフィールド`(location_type`)は、停留所と駅を区別するために使用される。
```
bracket in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_codeと` `stop_nameは`、通常、ライダー向けの情報を含みます。
+- 正確な位置は、座標`(stop_lat`と`stop_lon`) を使用して提供されます。
+- 6番目のフィールド`(location_type`)は、停留所と駅を区別するために使用される。
+- 最初のレコードはバス停に対応するため、`location_type=0と`なる。
```
Japanese character in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+- 6番目のフィールド`(location_type`)は、停留所と駅を区別するために使用される。
+- 最初のレコードはバス停に対応するため、`location_type=0と`なる。
+- 2番目のレコードは駅に対応し、`location_type=1`である。
+- ` location_typeの  `値の完全なリストは、[stops.txt](../../reference/stopstxt)見つけることができる。
```
Japanese character in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+
+経路を記述した後、各経路が提供するトリップを記述することができる。
+
+まず、[calendar.txt](../../reference/#calendartxt)xtを使用して、サービスのスパンを定義する必要があります。
```
extra xt after hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+    service_id,monday,tuesday,wednesday,thursday,friday,saturday,sunday,start_date,end_date
+    weekend_service,0,0,0,0,0,1,1,20220623,20220903
+
+ここでは、土日のみ運行するサービスを記述しているので、これらの日のフィールドには1が、残りの日のフィールドには0が入力されています。このサービスは、`start_dateと` `end_dateの`フィールドにあるように、2022年6月23日から2022年9月3日まで運行されます。
```
Japanese character in code format both times

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+    303-20670,weekend_service,60270566,"MAX ORANGE BRENTWOOD",1,3030027
+
+- [routes.txt](../../reference/#routestxt)のうち、MAX Orange に対応する`route_id`が記載されています。
+- [calendar.txt](../../reference/#calendartxt)xt のうち、週末に該当する`service_id`が記載されています。
```
extra xt after hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+- [calendar.txt](../../reference/#calendartxt)xt のうち、週末に該当する`service_id`が記載されています。
+- 各レコードには、各旅行に対して一意のID
+
+ヘッドサインのテキストは、バスの内部と外部の標識に通常表示されるもので、提供されます。
```
errant line break

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+
+- `direction_id`フィールドは、同じルートで異なる方向に向かうトリップを区別することができます。例えば、インバウンドのトリップとアウトバウンドのトリップ、あるいは南行きのトリップと北行きのトリップを区別することができます。
+  - この場合、Saddletowne方面のトリップは`direction_id=0`、Brentwood方面のトリップは`direction_id=1`です。direction_id の値には固有の意味はなく、ある進行方向と別の進行方向を指定するために使用されるだけである。
+- [shapes.txt](../../reference/#shapestxt)xtから、Saddletowne方面のMAX Orangeルートに対応する`shape_idが`1レコード目に、Brentwood方面のMAX Orangeルートに対応するものが2レコード目と3レコード目に記載されている
```
extra xt after shapes.txt 
Japanese character included in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+
+運行日の追加（特別日）や運行日の削除（休日運休など）を設定することができます。
+
+例えば、2022年7月17日（日）に予定されているサービスがない場合、サービスを2つに分けることで、[calendar.txtの](../../reference/#calendartxt) `weekend_serviceから`その日付を削除することができます。
```
Japanese character in hyperlink and in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.id.md:
```diff
@@ -0,0 +1,134 @@
+
+Misalnya, jika tidak ada layanan terjadwal pada hari Minggu 17 Juli 2022 - maka tanggal tersebut dapat dihapus dari `weekend_service` di [calendar.txt](../../reference/#calendartxt)  dengan memecah layanan menjadi dua:
+
+|  Melayani          | start      | end        |
```
Meant to translate?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.es.md:
```diff
@@ -0,0 +1,134 @@
+
+Por ejemplo, si no hay servicio programado el domingo 17 de julio de 2022 - entonces esa fecha se puede eliminar de `weekend_service` en [calendar.txt](../../reference/#calendartxt) txt dividiendo el servicio en dos:
+
+| Servicio           | start      | end        |
```
Meant to translate?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.de.md:
```diff
@@ -0,0 +1,134 @@
+
+Wenn zum Beispiel am Sonntag, dem 17. Juli 2022, kein planmäßiger Dienst stattfindet, kann dieses Datum aus `weekend_service` in [calendar.txt](../../reference/#calendartxt) entfernt werden, indem der Service in zwei Datensätze aufgeteilt wird:
+
+| Dienst             | start      | end        |
```
Meant to translate?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+
+例えば、2022年7月17日（日）に予定されているサービスがない場合、サービスを2つに分けることで、[calendar.txtの](../../reference/#calendartxt) `weekend_serviceから`その日付を削除することができます。
+
+| サービス               | 開始         | 終了         |
```
Meant to translate?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+| `weekend_service1` | `20220623` | `20220716` |
+| `weekend_service2` | `20220718` | `20220903` |
+
+しかし、`service_idが`2つに分割され、この分割が[trips.txtに](../../reference/#tripstxt)連鎖してしまうため、ファイルが複雑になってしまいます。代わりに、以下のように[calendar_dates.txtを](../../reference/#calendar_datestxt)使用することで、より簡単な方法でこれを行うことができます。
```
Japanese character formatted as code and included in trips.txt and calendar_dates.txt hyperlinks

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ja.md:
```diff
@@ -0,0 +1,134 @@
+    service_id,date,exception_type
+    weekend_service,20220623,2
+
+- `service_id` `weekend_serviceと`なる。
```
Japanese character formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ko.md:
```diff
@@ -0,0 +1,134 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone
+    CT,Calgary Transit,http://www.calgarytransit.com,America/Edmonton,,403-262-1000
+
+Calgary Transit은 AB 캘거리에서 LRT, BRT, 일반 버스 서비스, 보조 대중 교통 및 주문형 대중 교통을 운영합니다. 이 예에서는 두 개의 경로가 정의됩니다. 첫 번째는 버스이고 두 번째는 LRT입니다. 파일 사용[routes.txtという](../../reference/#routestxt) , 각 경로에는 고유한id , 사람이 쉽게 읽을 수 있도록 짧은 이름과 긴 이름이 있습니다.
```
Korean characters included in routes.txt hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ko.md:
```diff
@@ -0,0 +1,134 @@
+- 그만큼`service_id` \~에서[calendar.txt](../../reference/#calendartxt) 주말에 해당하는
+- 각 레코드에는 고유한id 각각trip
+
+헤드 사인text 버스 안팎의 표지판에 일반적으로 표시되는 것입니다.
```
Errant space
"text" on line 85 must be an error as well

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ko.md:
```diff
@@ -0,0 +1,134 @@
+
+헤드 사인text 버스 안팎의 표지판에 일반적으로 표시되는 것입니다.
+
+- 필드direction_id 서로 다른 방향으로 가는 동일한 경로의 여행을 구분할 수 있습니다. 예를 들어, 인바운드 여행과 아웃바운드 여행 또는 남쪽 여행과 북쪽 여행을 구별합니다.
```
direction_id not in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ko.md:
```diff
@@ -0,0 +1,134 @@
+
+다음과 같은 서비스에 대한 예외를 정의할 수 있습니다.added 휴무일(특정일) 또는 휴무일(예: 공휴일 휴무)
+
+예를 들어,scheduled 2022년 7월 17일 일요일 서비스 - 그러면 해당 날짜는 다음의 `weekend_service` 에서 제거될 수 있습니다.[calendar.txtの](../../reference/#calendartxt) 서비스를 둘로 나누면:
```
Korean character in hyperlink

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ko.md:
```diff
@@ -0,0 +1,134 @@
+
+예를 들어,scheduled 2022년 7월 17일 일요일 서비스 - 그러면 해당 날짜는 다음의 `weekend_service` 에서 제거될 수 있습니다.[calendar.txtの](../../reference/#calendartxt) 서비스를 둘로 나누면:
+
+|  서비스               | start      | end        |
```
Meant to be translated?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ko.md:
```diff
@@ -0,0 +1,134 @@
+| `weekend_service1` | `20220623` | `20220716` |
+| `weekend_service2` | `20220718` | `20220903` |
+
+그러나 이로 인해 파일이 복잡해집니다.`service_id` 두 개로 나뉘며 이 나누기는[trips.txtに](../../reference/#tripstxt) . 대신 다음을 사용하여 더 쉽게 수행할 수 있습니다.[calendar_dates.txtを](../../reference/#calendar_datestxt) 아래 그림과 같이:
```
Korean characters in both hyperlinks

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+O quinto campo`route_type`) é usado para diferenciar os tipos de rotas:
```
Missing opening bracket

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_id` é um identificador único
+- `stop_code` e `stop_name` normalmente contêm informações voltadas para o cavaleiro
+- A localização exata é fornecida usando coordenadas`(stop_lat` e `stop_lon`)
+- O sexto campo`(location_type`) é usado para diferenciar as paradas das estações
```
opening brackets formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+- O sexto campo`(location_type`) é usado para diferenciar as paradas das estações
+- O primeiro registro corresponde a uma parada de ônibus, portanto `local_type=0`
+- O segundo registro corresponde à estação, portanto `local_type=1`
+- Uma lista completa de valores para o ` location_type  `pode ser encontrada [aqui](../../reference/stopstxt)
```
space included in code format

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+- O `service_id` do [calendar.txt](../../reference/#calendartxt) que corresponde aos fins de semana é listado
+- Cada registro contém uma id única para cada trip.
+
+É fornecido o text do sinal principal, que é o que normalmente é exibido nas placas dentro e fora do ônibus
```
Errant space

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+  - Neste caso, as viagens em direção a Saddletowne têm uma `direction_id=0` e as viagens em direção a Brentwood têm uma `direction_id=1`. Os valores em direction_id não têm significado inerente, eles são usados apenas para atribuir uma direção de viagem em relação a outra
+- O `shape_id` de [shapes.txt](../../reference/#shapestxt) que corresponde à rota MAX Orange em direção a Saddletowne é listado para o primeiro registro e o da rota MAX Orange em direção a Brentwood é listado para o segundo e terceiro registros
+
+A `forma_id=3030026` corresponde ao MAX Orange towards Saddletowne. O arquivo abaixo inclui informações sobre os pontos que traçam o percurso, assim como a distância entre esses pontos. Com estas informações, é possível traçar a rota em um mapa para fins de planejamento ou análise da trip.
```
shape_id was translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+
+É possível definir exceções ao serviço como os dias de serviço adicionados (dias especiais) ou dias de serviço removidos (como nenhum serviço em feriados).
+
+Por exemplo, se não houver um serviço agendados no domingo 17 de julho de 2022 - então essa data pode ser removida do `serviço de fim de semana` no [calendar.txt](../../reference/#calendartxt), quebrando o serviço em dois:
```
weekend_service was translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.pt_BR.md:
```diff
@@ -0,0 +1,134 @@
+
+Por exemplo, se não houver um serviço agendados no domingo 17 de julho de 2022 - então essa data pode ser removida do `serviço de fim de semana` no [calendar.txt](../../reference/#calendartxt), quebrando o serviço em dois:
+
+| Serviço            | start      | end        |
```
meant to be translated?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ru.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+Пятое поле`route_type`) используется для различения типов маршрутов:
```
missing opening bracket

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ru.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_id` - уникальный идентификатор
+- `stop_code` и `stop_name` обычно содержат информацию, ориентированную на пассажиров.
+- Точное местоположение предоставляется с помощью координат`(stop_lat` и `stop_lon`).
+- Шестое поле`(location_type`) используется для отличия остановок от станций.
```
opening brackets formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ru.md:
```diff
@@ -0,0 +1,134 @@
+- Шестое поле`(location_type`) используется для отличия остановок от станций.
+- Первая запись соответствует автобусной остановке, следовательно, `location_type=0`
+- Вторая запись соответствует станции, следовательно, `location_type=1`
+- Полный список значений для ` location_type  `можно найти [здесь](../../reference/stopstxt)
```
space formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.ru.md:
```diff
@@ -0,0 +1,134 @@
+- Перечислен `service_id` из [calendar.txt](../../reference/#calendartxt), соответствующий выходным.
+- Каждая запись содержит уникальный id для каждой trip
+
+Представлен text заголовка, который обычно отображается на табличках внутри и снаружи автобуса.
```
errant space 
"trip" and "text" not being translated might be an error

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang,agency_phone
+    CT,Calgary Transit,http://www.calgarytransit.com,America/Edmonton,,403-262-1000
+
+卡尔加里公交公司在AB省卡尔加里市经营轻轨、快速公交、普通公交、准公交和按需公交。在这个例子中，定义了两条线路，第一条是公共汽车，第二条是轻轨。使用[routes.txt](../../reference/#routestxt)文件，每条路线都被分配了一个唯一的id，以及一个短名称和一个长名称，以便于人阅读。
```
"Calgary Transit" gets translated here

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+    agency_id,route_id,route_short_name,route_long_name,route_type,route_url,route_color,route_text_color
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
```
missing opening bracket

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+第五个字段`route_type`）是用来区分路线的类型。
+
+- 第一条是公共汽车，因此`路线_类型=3`
+- 第二条是轻轨，因此`路线_类型=0`
```
route_type gets translated here twice

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+- `stop_id`是一个唯一的标识符
+- `stop_code`和`stop_name`通常包含面向乘客的信息
+- 使用坐标`（stop_lat`和`stop_lon`）提供确切的位置。
+- 第六个字段`（location_type`）用于区分站点和车站。
```
spaces and opening brackets formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+- 第六个字段`（location_type`）用于区分站点和车站。
+- 第一条记录对应的是一个公交车站，因此`location_type=0`
+- 第二条记录对应的是车站，因此`location_type=1`。
+- 关于` location_type  `值的完整列表可以[在这里](../../reference/stopstxt)找到。
```
space formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+    303-20670,weekend_service,60270566,"MAX ORANGE BRENTWOOD",1,3030027
+
+- 从[routes.txt](../../reference/#routestxt)txt中列出了与MAX Orange对应的`route_id`。
+- 列出了[calendar.txt](../../reference/#calendartxt)txt中与周末相对应的`service_id`。
```
extra txt after both hyperlinks

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+- 列出了[calendar.txt](../../reference/#calendartxt)txt中与周末相对应的`service_id`。
+- 每条记录包含每个trip的唯一id
+
+提供了车头标志text，这就是通常显示在巴士内部和外部的标志上的内容。
```
errant line break
text and trip not translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+提供了车头标志text，这就是通常显示在巴士内部和外部的标志上的内容。
+
+- `direction_id`字段允许区分同一路线上不同方向的车次。例如，区分进站和出站的车次，或南行和北行的车次。
+  - 在这种情况下，往Saddletowne方向的车次，方向`_id=0`，往Brentwood方向的车次，`方向_id=1`。direction_id中的值没有内在的意义，它们只是用来指定一个旅行方向与另一个旅行方向。
```
direction_id being translated twice and not formatted properly twice

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+
+可以定义服务的例外情况，如ADDED服务日（特殊日子）或取消的服务日（如节假日不提供服务）。
+
+例如，如果2022年7月17日星期日没有SCHEDULED服务--那么该日期可以通过将服务一分为二从[calendar.txt](../../reference/#calendartxt)txt的`周末服务`中删除。
```
extra txt after hyperlink
weekend_service translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+
+例如，如果2022年7月17日星期日没有SCHEDULED服务--那么该日期可以通过将服务一分为二从[calendar.txt](../../reference/#calendartxt)txt的`周末服务`中删除。
+
+| 服务                 | start      | end        |
```
meant to be translated?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+    service_id,date,exception_type
+    weekend_service,20220623,2
+
+- `service_id` `周末服务`被列出。
```
weekend_service translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh.md:
```diff
@@ -0,0 +1,134 @@
+    weekend_service,20220623,2
+
+- `service_id` `周末服务`被列出。
+- 移除或ADDED服务的日期列在`日期`下（2022年7月17日）。
```
`date` translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+    CT,303-20670,303,MAX Orange Brentwood/Saddletowne,3,www.calgarytransit.com/content/transit/en/home/rider-information/max.html,#ff8000,#ffffff
+    CT,202-20666,202,Blue Line - Saddletowne/69 Street CTrain,0,www.calgarytransit.com/content/transit/en/home/rider-information/lrt-and-bus-station-maps.html,#ff0000,#ffffff
+
+第五個字段（route_type ) 用於區分路由類型：
```
`route_type` not formatted as code

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+
+## 停止
+
+在GTFS , 停靠站和車站使用文件進行描述stops.txt ，下面，第一個記錄中定義了一個公共汽車站，第二個記錄中定義了一個輕軌站。
```
stops.txt hyperlink missing

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+- 第二個是輕軌，因此`route_type=0`
+- 完整的值列表`route_type`可以在[這裡](../../reference/#routestxt)找到
+
+其餘字段包含其他信息，例如url特定於路線以及特定於機構的顏色以在地圖上表示服務。
```
"url"?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+
+此處描述了僅在周六和周日運行的服務，因此這些天的字段填充為 1，其余天的字段填充為零。該服務從 2022 年 6 月 23 日到 2022 年 9 月 3 日運行，如字段下方所示`start_date`和`end_date` 。
+
+在這個例子中，文件trips.txt描述了由上述 MAX Orange 路線提供服務的 3 次週末旅行。
```
trips.txt hyperlink missing

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+- 這`service_id`從[calendar.txt](../../reference/#calendartxt)列出對應於週末的
+- 每條記錄都包含一個唯一的id每個trip
+
+頭標text提供，這通常顯示在巴士內外的標誌上
```
errant line break
id, trip, and text not translated?

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+
+可以定義服務的例外情況，例如added服務日（特殊日）或取消服務日（如節假日不提供服務）。
+
+例如，如果沒有scheduled 2022 年 7 月 17 日星期日的服務 - 然後可以從`weekend_service`中刪除該日期[calendar.txt](../../reference/#calendartxt)通過將服務一分為二：
```
"scheduled" not translated

josee-sabourin commented on docs/schedule/examples/routes-stops-trips.zh_TW.md:
```diff
@@ -0,0 +1,134 @@
+
+例如，如果沒有scheduled 2022 年 7 月 17 日星期日的服務 - 然後可以從`weekend_service`中刪除該日期[calendar.txt](../../reference/#calendartxt)通過將服務一分為二：
+
+| 服務                 | start      | end        |
```
Meant to be translated?

josee-sabourin commented on docs/schedule/examples/translations.de.md:
```diff
@@ -0,0 +1,50 @@
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- In diesem Beispiel werden die Haltestellennamen aus [stops.txt](../../reference/#stopstxt) übersetzt, und die Datensätze in [stops.txt](../../reference/#stopstxt) werden durch eine `stop_id` identifiziert. Daher werden
+  - `Haltestellen` werden unter Tabellennamen aufgeführt (mit Bezug auf [stops.txt](../../reference/#stopstxt))
```
`stops` tranaslated

josee-sabourin commented on docs/schedule/examples/translations.es.md:
```diff
@@ -0,0 +1,50 @@
+    S8815040,,Bruxelles-Ouest,,50.8485600,4.32104000,,,1,,
+    S8821006,,Anvers-Central,,51.2172000,4.42109800,,,1,,
+
+El archivo [translations.txt](../../reference/#translationstxt).txt se utiliza entonces para traducir los nombres de las estaciones del idioma por defecto de la agencia (francés en este caso) al holandés.
```
extra .txt after hyperlink

josee-sabourin commented on docs/schedule/examples/translations.es.md:
```diff
@@ -0,0 +1,50 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang
+    NMBS/SNCB,NMBS/SNCB,http://www.belgiantrain.be/,Europe/Brussels,fr
+
+Como el idioma de la agencia está configurado en francés, los nombres de las estaciones aparecen en francés en [stops.txt](../../reference/#stopstxt) archivo stops.txt.
```
something is off here, stops.txt appears twice

josee-sabourin commented on docs/schedule/examples/translations.es.md:
```diff
@@ -0,0 +1,50 @@
+    table_name,field_name,record_id,language,translation
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- En este ejemplo, los nombres de las estaciones de [stops.txt](../../reference/#stopstxt).txt se traducen y los registros de [stops.txt](../../reference/#stopstxt) se identifican con un `stop_id`. Por lo tanto
```
extra .txt after first hyperlink

josee-sabourin commented on docs/schedule/examples/translations.es.md:
```diff
@@ -0,0 +1,50 @@
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- En este ejemplo, los nombres de las estaciones de [stops.txt](../../reference/#stopstxt).txt se traducen y los registros de [stops.txt](../../reference/#stopstxt) se identifican con un `stop_id`. Por lo tanto
+  - las`stops` aparecen en los nombres de las tablas (en referencia a [stops.txt](../../reference/#stopstxt).txt)
```
extra .txt

josee-sabourin commented on docs/schedule/examples/translations.es.md:
```diff
@@ -0,0 +1,50 @@
+- El nombre se traduce al neerlandés (nl)
+- Por último, el nombre traducido aparece en la Translation
+
+Existe otra forma de traducir los nombres en GTFS utilizando el archivo [translations.txt](../../reference/#translationstxt), donde se utiliza el campo `field_value` en lugar de `record_id`. En este caso, el nombre de la estación se utiliza para encontrar el registro que se debe traducir desde [stops.txt](../../reference/#stopstxt).txt.
```
extra .txt after stops.txt

josee-sabourin commented on docs/schedule/examples/translations.id.md:
```diff
@@ -0,0 +1,50 @@
+    table_name,field_name,record_id,language,translation
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- Dalam contoh ini, nama stasiun dari[stops.txt](../../reference/#stopstxt) diterjemahkan, dan dicatat dalamstops.txt diidentifikasi oleh a `stop_id` . Karena itu:
```
need space before both stops.txt
second stops.txt is not formatted as a hyperlink

josee-sabourin commented on docs/schedule/examples/translations.id.md:
```diff
@@ -0,0 +1,50 @@
+- Nama diterjemahkan ke dalam bahasa Belanda (nl)
+- Terakhir, nama yang diterjemahkan tercantum di bawahTranslation
+
+Ada cara lain untuk menerjemahkan namaGTFS menggunakan berkas[translations.txt](../../reference/#translationstxt) , lapangan mana`field_value` digunakan sebagai gantinya`record_id` . Dalam hal ini, nama stasiun digunakan untuk menemukan rekaman yang akan diterjemahkan[stops.txt](../../reference/#stopstxt) .
```
need spaces before both hyperlinks

josee-sabourin commented on docs/schedule/examples/translations.ja.md:
```diff
@@ -0,0 +1,50 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang
+    NMBS/SNCB,NMBS/SNCB,http://www.belgiantrain.be/,Europe/Brussels,fr
+
+代理店言語がフランス語に設定されているので、[stops.txtには](../../reference/#stopstxt)駅名がフランス語でリストアップされています。
```
Japanese characters in hyperlink

josee-sabourin commented on docs/schedule/examples/translations.ja.md:
```diff
@@ -0,0 +1,50 @@
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- この例では、[stops.txt](../../reference/#stopstxt)の駅名が翻訳され、[stops.txt](../../reference/#stopstxt)のレコードが`stop_id` で識別されます。そのため
+  - `stopsは`テーブル名の下に記載されています（[stops.txtを](../../reference/#stopstxt)参照）。
```
Japanese characters in code format and hyperlink

josee-sabourin commented on docs/schedule/examples/translations.ja.md:
```diff
@@ -0,0 +1,50 @@
+- この例では、[stops.txt](../../reference/#stopstxt)の駅名が翻訳され、[stops.txt](../../reference/#stopstxt)のレコードが`stop_id` で識別されます。そのため
+  - `stopsは`テーブル名の下に記載されています（[stops.txtを](../../reference/#stopstxt)参照）。
+  - `stop_nameは` `field_nameの`下に記載される（駅名が翻訳されるため）。
+  - フランス語から翻訳される駅名の`stop_idは` `record_idの`下に記載されている（この場合、Bruxelles-Ouestの`stop_id`）。
```
Japanese characters in code format

josee-sabourin commented on docs/schedule/examples/translations.ja.md:
```diff
@@ -0,0 +1,50 @@
+- 名称はオランダ語（nl）に翻訳されています。
+- 最後に翻訳された名称を翻訳に記載
+
+GTFSで名前を翻訳するには、[translations.txtという](../../reference/#translationstxt)ファイルを使う方法もあり、ここでは`record_idの`代わりに`field_valueという`フィールドが使われる。この場合、[stops.txt](../../reference/#stopstxt)xtから翻訳するレコードを探すのに駅名が使われる。
```
Japanese characters in code format and hyperlink
Extra xt after stops.txt

josee-sabourin commented on docs/schedule/examples/translations.ko.md:
```diff
@@ -0,0 +1,50 @@
+    agency_id,agency_name,agency_url,agency_timezone,agency_lang
+    NMBS/SNCB,NMBS/SNCB,http://www.belgiantrain.be/,Europe/Brussels,fr
+
+기관 언어가 프랑스어로 설정되어 있으므로 스테이션 이름은 프랑스어로 나열됩니다.[stops.txtには](../../reference/#stopstxt) .
```
Korean characters in hyperlink

josee-sabourin commented on docs/schedule/examples/translations.ko.md:
```diff
@@ -0,0 +1,50 @@
+    table_name,field_name,record_id,language,translation
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- 이 예에서 스테이션 이름은[stops.txt](../../reference/#stopstxt) 번역되고 기록이stops.txt 에 의해 식별됩니다`stop_id` . 그러므로:
```
second stops.txt not formatted as hyperlink

josee-sabourin commented on docs/schedule/examples/translations.ko.md:
```diff
@@ -0,0 +1,50 @@
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- 이 예에서 스테이션 이름은[stops.txt](../../reference/#stopstxt) 번역되고 기록이stops.txt 에 의해 식별됩니다`stop_id` . 그러므로:
+  - `stops` 은 테이블 이름 아래에 나열됩니다(참조:[stops.txtを](../../reference/#stopstxt) )
```
Korean character in hyperlink

josee-sabourin commented on docs/schedule/examples/translations.ko.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name` 이 아래에 나열됩니다.`field_name` (역 이름이 번역되기 때문에)
+  - 그만큼`stop_id` 프랑스어에서 번역할 역 이름은 아래에 나열되어 있습니다.`record_id` (이 경우,`stop_id` Bruxelles-Ouest의 경우)
+- 이름은 네덜란드어로 번역됩니다(nl).
+- 마지막으로 번역된 이름이 아래에 나열됩니다.Translation
```
translation not translated

josee-sabourin commented on docs/schedule/examples/translations.ko.md:
```diff
@@ -0,0 +1,50 @@
+- 이름은 네덜란드어로 번역됩니다(nl).
+- 마지막으로 번역된 이름이 아래에 나열됩니다.Translation
+
+이름을 번역하는 또 다른 방법이 있습니다.GTFS 파일을 사용하여[translations.txtという](../../reference/#translationstxt) , 여기서 필드`field_value` 대신 사용됩니다`record_id` . 이 경우 스테이션 이름은 번역할 레코드를 찾는 데 사용됩니다.[stops.txt](../../reference/#stopstxt) .
```
Korean character in hyperlink

josee-sabourin commented on docs/schedule/examples/translations.zh.md:
```diff
@@ -0,0 +1,50 @@
+    table_name,field_name,record_id,language,translation
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- 在这个例子中，[stops.txt](../../reference/#stopstxt)txt中的车站名称被翻译，[stops.txt](../../reference/#stopstxt)中的记录由一个`stop_id`来识别。因此。
```
extra txt after first hyperlink

josee-sabourin commented on docs/schedule/examples/translations.zh.md:
```diff
@@ -0,0 +1,50 @@
+    stops,stop_name,S8815040,nl,Brussel-West
+
+- 在这个例子中，[stops.txt](../../reference/#stopstxt)txt中的车站名称被翻译，[stops.txt](../../reference/#stopstxt)中的记录由一个`stop_id`来识别。因此。
+  - `站`名列在表名下（参照[stops.txt](../../reference/#stopstxt)）。
```
`stops` translated

josee-sabourin commented on docs/schedule/examples/translations.zh.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name`列在`field_name`下（因为站名将被翻译）。
+  - 将被翻译成法语的站名的`stop_id`被列在`record_id`下（在这个例子中，Bruxelles-Ouest的`stop_id`）。
+- 该名称被翻译成荷兰语（nl）。
+- 最后，翻译后的名称被列在Translation下
```
Translation not translated

josee-sabourin commented on docs/schedule/examples/translations.zh_TW.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name`列在`field_name` （因為站名會被翻譯）
+  - 這`stop_id`要從法語翻譯的站名列在下面`record_id` （在這種情況下，`stop_id`西布魯塞爾）
+- 該名稱被翻譯成荷蘭語 (nl)
+- 最後，翻譯後的名稱列在Translation
```
translation not translated

josee-sabourin commented on docs/schedule/examples/translations.pt_BR.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name` está listado em `field_name` (uma vez que os nomes das estações serão traduzidos)
+  - O `stop_id` para o nome da estação a ser traduzido do francês está listado em `record_id` (neste caso, o `stop_id` para Bruxelles-Ouest)
+- O nome é traduzido para o holandês (nl)
+- Finalmente, o nome traduzido está listado em Translation
```
translation not translated

josee-sabourin commented on docs/schedule/examples/translations.id.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name` terdaftar di bawah`field_name` (karena nama stasiun akan diterjemahkan)
+  - Itu`stop_id` untuk nama stasiun yang akan diterjemahkan dari bahasa Prancis tercantum di bawah`record_id` (dalam hal ini,`stop_id` untuk Bruxelles-Ouest)
+- Nama diterjemahkan ke dalam bahasa Belanda (nl)
+- Terakhir, nama yang diterjemahkan tercantum di bawahTranslation
```
Translation not translated

josee-sabourin commented on docs/schedule/examples/translations.es.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name` aparece bajo `field_name` (ya que los nombres de las estaciones se traducirán)
+  - El `stop_id` del nombre de la estación que se va a traducir del francés aparece bajo `record_id` (en este caso, el `stop_id` de Bruxelles-Ouest)
+- El nombre se traduce al neerlandés (nl)
+- Por último, el nombre traducido aparece en la Translation
```
translation not translated

josee-sabourin commented on docs/schedule/examples/translations.de.md:
```diff
@@ -0,0 +1,50 @@
+  - `stop_name` wird unter `field_name` aufgeführt (da die Namen der Haltestellen übersetzt werden)
+  - Die `stop_id` für den aus dem Französischen zu übersetzenden Haltestellennamen wird unter `record_id` aufgeführt (in diesem Fall die `stop_id` für Bruxelles-Ouest)
+- Der Name wird ins Niederländische (nl) übersetzt.
+- Schließlich wird der übersetzte Name unter Translation aufgeführt
```
translation not translated

josee-sabourin commented on docs/schedule/process.de.md:
```diff
@@ -0,0 +1,336 @@
+1. Erstellen Sie einen Git-Zweig mit der Aktualisierung aller relevanten Teile der Protokolldefinition, der Spezifikation und der Dokumentationsdateien (mit Ausnahme der Übersetzungen).
+1. Erstellen Sie einen Pull-Request auf [https://github.com/google/transit.](https://github.com/google/transit) Der Pull Request muss eine ausführliche Beschreibung des Patches enthalten. Der Ersteller des Pull Requests wird zum _Advocate_.
+1. Sobald der Pull Request registriert ist, muss er von seinem Befürworter in der [GTFS Changes Mailingliste](https://groups.google.com/forum/#!forum/gtfs-changes) angekündigt werden, einschließlich eines Links zum Pull Request. Nach der Ankündigung wird der Pull Request als Vorschlag betrachtet. Die Pull-Anfrage sollte auch so bearbeitet werden, dass sie einen Link zu der Google Groups-Ankündigu ...
+   * Da der Befürworter ein Mitwirkender ist, muss er die [Lizenzvereinbarung für Mitwirkende](../CONTRIBUTING.md) unterzeichnen, bevor die Pull-Anfrage angenommen werden kann.
```
not sure if Contributor License Agreement was meant to be translated

josee-sabourin commented on docs/schedule/process.de.md:
```diff
@@ -0,0 +1,336 @@
+
+**März 24, 2020**<br/>
+
+* text Feld definiert und `tts_stop_name` zu `stops.txt` hinzugefügt. Siehe [Diskussion](https://github.com/google/transit/pull/49).
```
something looks off here

josee-sabourin commented on docs/schedule/process.de.md:
```diff
@@ -0,0 +1,336 @@
+
+**14. September 2018**<br/>
+
+* Konzept "Bedingt erforderlich" hinzugefügt. Siehe [Diskussion](https://github.com/google/transit/pull/100).
```
Not sure if "Conditionally Required" was meant to be translated, it isn't above

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**Los feeds deberían ser fáciles** de<br/>analizarLos lectores de feeds deberían poder extraer la información que buscan con el menor trabajo posible. Los cambios y adiciones al feed deberían ser lo más útiles posible, para minimizar el número de rutas de código que los lectores del feed tienen que implementar. (Sin embargo, debería darse prioridad a facilitar la creación, ya que al final habrá m ...
+
+La especificación**trata sobre la información a los pasajeros GTFS**<br/>se ocupa principalmente de la información a los pasajeros. Es decir, la especificación debería incluir información que pueda ayudar a potenciar las herramientas para los pasajeros, en primer lugar. Existe potencialmente una gran cantidad de información orientada a las operaciones que las agencias de transporte pueden querer  ...
```
title formatting issues

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+La especificación**trata sobre la información a los pasajeros GTFS**<br/>se ocupa principalmente de la información a los pasajeros. Es decir, la especificación debería incluir información que pueda ayudar a potenciar las herramientas para los pasajeros, en primer lugar. Existe potencialmente una gran cantidad de información orientada a las operaciones que las agencias de transporte pueden querer  ...
+
+**Al**<br/>añadir funciones a la especificación, queremos evitar cambios que invaliden los feeds existentes. No queremos crear más trabajo para los editores de feeds existentes hasta que quieran añadir capacidades a sus feeds. Además, siempre que sea posible, queremos que los analizadores existentes puedan seguir leyendo las partes antiguas de los nuevos feeds.<br/>
```
title issues

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**Los feeds deberían ser fáciles de crear y**<br/>editarElegimos CSV como base para la especificación porque es fácil de ver y editar utilizando programas de hojas de cálculo y editores de text, lo que resulta útil para las agencias más pequeñas. También es fácil de generar a partir de la mayoría de lenguajes de programación y bases de datos, lo que es bueno para los editores de feeds más grandes ...
+
+**Los feeds deberían ser fáciles** de<br/>analizarLos lectores de feeds deberían poder extraer la información que buscan con el menor trabajo posible. Los cambios y adiciones al feed deberían ser lo más útiles posible, para minimizar el número de rutas de código que los lectores del feed tienen que implementar. (Sin embargo, debería darse prioridad a facilitar la creación, ya que al final habrá m ...
```
title formatting issues

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+Con el fin de preservar la visión original de GTFS, se han establecido una serie de principios rectores a tener en cuenta a la hora de ampliar la especificación:
+
+**Los feeds deberían ser fáciles de crear y**<br/>editarElegimos CSV como base para la especificación porque es fácil de ver y editar utilizando programas de hojas de cálculo y editores de text, lo que resulta útil para las agencias más pequeñas. También es fácil de generar a partir de la mayoría de lenguajes de programación y bases de datos, lo que es bueno para los editores de feeds más grandes ...
```
title formatting issues

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**4 de enero de 2021**<br/>
+
+* Aclarada la descripción de `stop_id.stop_id`. Ver [discusión](https://github.com/google/transit/pull/258).
```
should be `stop_times.stop_id`

josee-sabourin commented on docs/schedule/process.de.md:
```diff
@@ -0,0 +1,336 @@
+
+**2. Februar 2015**<br/>
+
+* stop_times.txt xt 'timepoint' Vorschlag zu spec hinzugefügt: [Diskussion](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/Ah-J9JP2rJY)
```
at this point down, even in the english version, nothing is formatted as code

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**5 de febrero de 2020**<br/>
+
+* Añadidos `route_types de` trolebús y monorraíl. Véase [discusión](https://github.com/google/transit/pull/174).
```
code text format error

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**14 de septiembre de 2018**<br/>
+
+* Añadido el concepto "Condicionalmente obligatorio". Véase [discusión](https://github.com/google/transit/pull/100).
```
Conditionally required translated

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**2 de febrero de 2015**<br/>
+
+* Añadida la propuesta stop_times.txt.txt 'timepoint' a las especificaciones: [discusión](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/Ah-J9JP2rJY)
```
extra .txt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**17 de febrero de 2014**<br/>
+
+* Añadida la propuesta trips.txt.txt 'bikes_allowed' a la especificación: [discusión](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/rEiSeKNc4cs)
```
extra .txt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+**30 de marzo de 2009**<br/>
+
+* Una nueva sección sobre cómo hacer público un feed de tránsito. Esto no se discutió previamente en el grupo, porque no era estrictamente un cambio en cómo se interpretan o escriben los datos. Sin embargo, algunos de los miembros de Google pensaron que sería informativo incluir un debate sobre los usos de GTFS ajenos a Google, ya que cada vez hay más aplicaciones que pueden utilizar datos GTFS.
+* [Aclaraciones](https://groups.google.com/forum/#!topic/gtfs-changes/03qz5aTA2mk) sobre el formato CSV: discusión
```
I think this hyperlink should be on discusión and not Aclaraciones

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+**6 de agosto de 2008**<br/>
+
+* Añadido el archivo transfers.txt, que permite a los editores de feeds proporcionar pistas sobre el comportamiento de transferencia preferido ([propuesta original](https://groups.google.com/forum/#!topic/gtfs-changes/cL1E4oKKpKw)) .
+* Se han añadido los campos location_type y parent_station a stops.txt.txt, lo que permite agrupar las paradas en estaciones ([propuesta original](https://groups.google.com/forum/#!topic/gtfs-changes/ScGAyZ9a_yw)).
```
extra .txt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+**29 de febrero de 2008**<br/>
+
+* Se ha añadido el campo stop_code en stops.txt para permitir la especificación de códigos de parada orientados al usuario ([propuesta original](https://groups.google.com/forum/#!topic/gtfs-changes/k9A95fYZexc)).
+* Aclaradas las descripciones de route_short_name y route_long_name en routes.txtxt
```
extra xt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**28 de febrero de 2007**<br/>
+
+* Adición de frequencies.txt.txt para soporte de Schedule basados en headway.
```
extra .txt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+* Adición de frequencies.txt.txt para soporte de Schedule basados en headway.
+* Ahora se permiten múltiples agencias en el mismo feed. También se ha añadido un nuevo campo agency_id en agencies.txt y routes.txt que permite especificar qué ruta es operada por qué agencia.
+* Adición de URL por ruta y por parada.
+* Adición del campo direction_id en trips.txtxt.
```
extra xt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+* Ahora se permiten múltiples agencias en el mismo feed. También se ha añadido un nuevo campo agency_id en agencies.txt y routes.txt que permite especificar qué ruta es operada por qué agencia.
+* Adición de URL por ruta y por parada.
+* Adición del campo direction_id en trips.txtxt.
+* Posibilidad de cambiar las cabeceras a mitad de trayecto añadiendo el campo stop_headsign en stop_times.txt.txt.
```
extra .txt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+* Adición de URL por ruta y por parada.
+* Adición del campo direction_id en trips.txtxt.
+* Posibilidad de cambiar las cabeceras a mitad de trayecto añadiendo el campo stop_headsign en stop_times.txt.txt.
+* Soporte para colores de ruta con la adición de route_color opcional y route_text_color en routes.txt.txt.
```
extra .txt

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**29 de noviembre de 2006**<br/>
+
+* Añadida la compatibilidad con la información de Shape trip a través de shapes.txt.txt
```
extra .txt

josee-sabourin commented on docs/schedule/process.fr.md:
this translation of CLI to CCT does not line up with above translation of "accord de licence du contributeur"

josee-sabourin commented on docs/schedule/process.fr.md:
title format error

josee-sabourin commented on docs/schedule/process.fr.md:
stop_name translated

josee-sabourin commented on docs/schedule/process.fr.md:
conditionally required translated

josee-sabourin commented on docs/schedule/process.fr.md:
missing bullet point
extra .txt

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+     - Jika advokat melanjutkan pengerjaan proposal, maka pemungutan suara baru dapat dilakukan kapan sajatime .
+  1. Setiap pull request yang tidak aktif selama 30 hari kalender akan ditutup. Saat pull request ditutup, proposal terkait dianggap ditinggalkan. Advokat dapat membuka kembali pull request kapan sajatime jika mereka ingin melanjutkan atau mempertahankan percakapan.
+  1. Jika proposal diterima:
+     - Google berkomitmen untuk menggabungkan versi pull request yang telah dipilih (asalkan kontributor telah menandatangani [PKB](../CONTRIBUTING.md) ), dan melakukan pull request dalam waktu 5 hari kerja.
```
PKB doesn't align with translation at line 17

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **13 Mei 2020**<br/>
+
+  * Menambahkan `continuous_pickup` dan `continuous_drop_off` keroutes.txt danstop_times.txt . Berubahshape_id dari "Optional" ke "Conditionally required". Lihat [diskusi](https://github.com/google/transit/pull/208) .
```
routes.txt, stop_times.txt and shape_id not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **25 Mei 2020**<br/>
+
+  * Ditetapkan pathways.txt ,levels.txt dan attributions.txt sebagai tabel yang dapat diterjemahkan. Menambahkan rekomendasi untuk menerjemahkan nilai `signposted_as` multibahasa. Lihat [diskusi](https://github.com/google/transit/pull/220) .
```
pathways.txt, levels.txt, attributions.txt not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **24 Maret 2020**<br/>
+
+  * Ditetapkantext -to-speech dan menambahkan `tts_stop_name` kestops.txt . Lihat [diskusi](https://github.com/google/transit/pull/49) .
```
stops.txt not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **9 Januari 2020**<br/>
+
+  * Ditambahkantranslations.txt . Lihat [diskusi](https://github.com/google/transit/pull/180) .
```
translations.txt not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **26 Desember 2019**<br/>
+
+  * Definisi yang diperbarui untuk trem kabel dan lift udara masukroute_type . Lihat [diskusi](https://github.com/google/transit/pull/186) .
```
route_type not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **20 Desember 2019**<br/>
+
+  * Ditambahkan attributions.txt . Lihat [diskusi](https://github.com/google/transit/pull/192) .
```
attributions.txt not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  * Ditambahkanarrival dandeparturetime praktik terbaik. Lihat [diskusi](https://github.com/google/transit/pull/165) .
+  * Menambahkan praktik terbaik headsign. Lihat [diskusi](https://github.com/google/transit/pull/167) .
+  * Ditambahkanstop_id praktik terbaik. Lihat [diskusi](https://github.com/google/transit/pull/169) .
```
stop_id not formatted as code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **4 April 2019**<br/>
+
+  * Menambahkan kolom `platform_code` di stops.txt . Lihat [diskusi](https://github.com/google/transit/pull/146) .
```
stops.txt not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **27 Maret 2019**<br/>
+
+  * Ditambahkan pathways.txt dan levels.txt . Lihat [diskusi](https://github.com/google/transit/pull/143) .
```
pathways.txt and levels.txt not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **14 September 2018**<br/>
+
+  * Menambahkan konsep "Diperlukan bersyarat". Lihat [diskusi](https://github.com/google/transit/pull/100) .
```
conditionally required translated

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **22 Agustus 2018**<br/>
+
+  * Menambahkan bidang `feed_contact_email` dan `feed_contact_url` difeed_info.txt mengajukan. Lihat [diskusi](https://github.com/google/transit/pull/31) .
```
feed_info.txt not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **11 Desember 2017**<br/>
+
+  * Menambahkan `route_sort_order` keroutes.txt . Lihat [diskusi](https://github.com/google/transit/pull/83) .
```
routes.txt not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **7 Februari 2017**<br/>
+
+  * Hubungan yang diklarifikasi dari `block_id` danservice_id . Lihat [diskusi](https://github.com/google/transit/pull/44) .
```
service_id not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  * Hubungan yang diklarifikasi dari `block_id` danservice_id . Lihat [diskusi](https://github.com/google/transit/pull/44) .
+  * Diklarifikasi bahwa layanan berbasis frekuensi dimulai padavehicledeparture . Lihat [diskusi](https://github.com/google/transit/pull/42) .
+  * Deskripsi yang diklarifikasi daristop_id dan `stop_code` . Lihat [diskusi](https://github.com/google/transit/pull/40) .
```
stop_id not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **11 Desember 2017**<br/>
+
+  * Menambahkan bidang `route_sort_order` diroutes.txt mengajukan. Lihat [diskusi](https://github.com/google/transit/pull/83) .
```
routes.txt not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **3 Februari 2016**<br/>
+
+  * Menambahkan `agency_email` keagency.txt proposal untuk spesifikasi: [diskusi](<https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/aezjQsriLYA)
```
agency.txt not code

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **2 Februari 2015**<br/>
+
+  * Ditambahkanstop_times.txt proposal 'titik waktu' untuk spesifikasi: [diskusi](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/Ah-J9JP2rJY>)
```
generally at this point onwards in this file spaces are needed before file or field names

josee-sabourin commented on docs/schedule/process.ja.md:
```diff
@@ -0,0 +1,336 @@
+
+GTFSの当初のビジョンを維持するために、仕様を拡張する際に考慮すべきいくつかの指導原則が確立されています。
+
+**フィードの作成と編集が容易**であること<br/>CSVを仕様のベースとして選んだのは、表計算ソフトやテキストエディタで簡単に表示・編集できるため、小規模な機関にとって<br/>便利だからです。<br/>また、ほとんどのプログラミング言語やデータベースから簡単に生成できるため、大規模なフィードを発行する場合にも適しています。<br/>
```
title formatting error

josee-sabourin commented on docs/schedule/process.ja.md:
```diff
@@ -0,0 +1,336 @@
+
+**旅客情報についての仕様であることGTFSは**、<br/>旅客情報を主な対象として<br/>います。<br/>つまり、ライダーのためのパワーツールに役立つ情報を、まず第一に仕様に盛り込むべきである。交通機関がシステム間で内部的に伝送したいと考える運用志向の情報は、潜在的に大量に存在する可能性があります。GTFSはそのためのものではなく、より適切な他のオペレーション指向のデータ標準が存在する可能性がある。<br/>
+
+**仕様の変更は後方互換性を**保つこと<br/>仕様に機能を追加<br/>する**場合**、<br/>既存のフィードを無効に<br/>するような変更は避けたいと考えています。<br/>既存のフィード発行者がフィードに機能を追加するまでの作業を増やしたくありません。また、可能な限り、既存のパーサーが新しいフィードの古い部分を読み続けられるようにしたいものです。<br/>
```
title formatting error

josee-sabourin commented on docs/schedule/process.ja.md:
```diff
@@ -0,0 +1,336 @@
+
+**仕様の変更は後方互換性を**保つこと<br/>仕様に機能を追加<br/>する**場合**、<br/>既存のフィードを無効に<br/>するような変更は避けたいと考えています。<br/>既存のフィード発行者がフィードに機能を追加するまでの作業を増やしたくありません。また、可能な限り、既存のパーサーが新しいフィードの古い部分を読み続けられるようにしたいものです。<br/>
+
+**投機的な機能は推奨さ**れない<br/>新しい機能を追加するたびに、フィードの作成と読み込みが複雑に<br/>なります。<br/>したがって、有用であることが分かっている機能のみを追加するように注意したいと思います。理想的には、どのような提案であっても、その新機能を使用する実際の交通システムのデータを生成し、それを読み込んで表示するソフトウェアを作成することによって、テストされている必要があります。GTFSは、公式のパーサーやバリデーターが無視するような列やファイルを追加することで、フォーマットの拡張を容易に行えることに留意してください。<br/>
```
title formatting error

josee-sabourin commented on docs/schedule/process.ja.md:
```diff
@@ -0,0 +1,336 @@
+
+**2018年9月14日（金**<br/>
+
+* 条件付き必須 "の概念を追加。[説明を](https://github.com/google/transit/pull/100)参照。
```
conditionally required translated

josee-sabourin commented on docs/schedule/process.ja.md:
```diff
@@ -0,0 +1,336 @@
+
+**2012年10月15日**<br/>
+
+trips.txt の 'wheelchair_accessible' 提案を仕様に追加：[議論](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.de.md:
```diff
@@ -0,0 +1,336 @@
+
+**Oktober 15, 2012**<br/>
+
+trips.txt wheelchair_accessible' Vorschlag zur Spezifikation hinzugefügt: [Diskussion](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.es.md:
```diff
@@ -0,0 +1,336 @@
+
+**15 de octubre de 2012**<br/>
+
+Añadida la propuesta trips.txt.txt 'wheelchair_accessible' a la especificación: [discusión](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.id.md:
```diff
@@ -0,0 +1,336 @@
+
+  **15 Oktober 2012**<br/>
+
+  Ditambahkantrips.txt 'wheelchair_accessible ' proposal untuk spesifikasi: [diskusi](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin reviewed (commented):
More than I expected, will finish the rest tomorrow

josee-sabourin reviewed (commented):
More than I expected, will finish the rest tomorrow

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2021년 1월 4일**<br/>
+
+  * `stop_times. stop_id`stop_id . [토론](https://github.com/google/transit/pull/258) 을 참조하십시오.
```
duplicate stop.id

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2020년 5월 25일**<br/>
+
+  * 한정된pathways.txt ,levels.txt 그리고attributions.txt 번역 가능한 테이블로. 다국어 `signposted_as` 값 번역에 대한 권장 사항이 추가되었습니다. [토론](https://github.com/google/transit/pull/220) 을 참조하십시오.
```
pathways.txt, levels.txt, attributions.txt not formatted as code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2020년 5월 13일**<br/>
+
+  * 에 `continuous_pickup` 및 `continuous_drop_off` 를 추가했습니다.routes.txt 그리고stop_times.txt . 변경됨shape_id "선택사항"에서 "조건부 필수"로. [토론](https://github.com/google/transit/pull/208) 을 참조하십시오.
```
routes.txt, stop_times.txt, shape_id not formatted as code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2020년 3월 24일**<br/>
+
+  * 한정된text -to-speech 필드에 `tts_stop_name` 을 추가했습니다.stops.txt . [토론](https://github.com/google/transit/pull/49) 을 참조하십시오.
```
stops.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2020년 1월 9일**<br/>
+
+  * 추가됨translations.txt . [토론](https://github.com/google/transit/pull/180) 을 참조하십시오.
```
translations.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2019년 12월 26일**<br/>
+
+  * 케이블 트램 및 공중 리프트에 대한 업데이트된 정의route_type . [토론](https://github.com/google/transit/pull/186) 을 참조하십시오.
```
route_type not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2019년 12월 20일**<br/>
+
+  * 추가됨attributions.txt . [토론](https://github.com/google/transit/pull/192) 을 참조하십시오.
```
attributions.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2019년 7월 9일**<br/>
+
+  * 추가됨arrival 그리고departuretime 모범 사례. [토론](https://github.com/google/transit/pull/165) 을 참조하십시오.
```
departure time not translated

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  * 추가됨arrival 그리고departuretime 모범 사례. [토론](https://github.com/google/transit/pull/165) 을 참조하십시오.
+  * 헤드 사인 모범 사례를 추가했습니다. [토론](https://github.com/google/transit/pull/167) 을 참조하십시오.
+  * 추가됨stop_id 모범 사례. [토론](https://github.com/google/transit/pull/169) 을 참조하십시오.
```
stop_id not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2019년 6월 25일**<br/>
+
+  * 명확한 관계Shape 포인트와 스톱. [토론](https://github.com/google/transit/pull/39) 을 참조하십시오.
```
shape not translated

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2019년 4월 4일**<br/>
+
+  * 에 추가된 `platform_code` 필드stops.txt . [토론](https://github.com/google/transit/pull/146) 을 참조하십시오.
```
stops.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2019년 3월 27일**<br/>
+
+  * 추가됨pathways.txt 그리고levels.txt . [토론](https://github.com/google/transit/pull/143) 을 참조하십시오.
```
pathways.txt, levels.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2018년 9월 14일**<br/>
+
+  * "조건부 필수" 개념이 추가되었습니다. [토론](https://github.com/google/transit/pull/100) 을 참조하십시오.
```
Conditionally required translated

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2018년 8월 22일**<br/>
+
+  * `feed_contact_email` 및 `feed_contact_url` 필드를feed_info.txt 파일. [토론](https://github.com/google/transit/pull/31) 을 참조하십시오.
```
feed_info.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2017년 12월 11일**<br/>
+
+  * `route_sort_order` 를 다음에 추가했습니다.routes.txt . [토론](https://github.com/google/transit/pull/83) 을 참조하십시오.
```
routes.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+  **2017년 2월 7일**<br/>
+
+  * `block_id` 와 명확한 관계service_id . [토론](https://github.com/google/transit/pull/44) 을 참조하십시오.
+  * 빈도 기반 서비스가 다음에서 시작됨을 명확히 함vehicledeparture . [토론](https://github.com/google/transit/pull/42) 을 참조하십시오.
```
vehicle departure not translated

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2017년 12월 11일**<br/>
+
+  * `route_sort_order` 필드를 추가했습니다.routes.txt 파일. [토론](https://github.com/google/transit/pull/83) 을 참조하십시오.
```
routes.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2016년 9월 2일**<br/>
+
+  * 추가할 업데이트된 설명서agency_id 아래에fare_attributes.txt . [토론](https://github.com/google/transit/pull/27) 을 참조하십시오.
```
agency_id, fare_attributes.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2016년 2월 3일**<br/>
+
+  * 다음에 `agency_email` 을 추가했습니다.agency.txt 사양에 대한 제안: [토론](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/aezjQsriLYA)
```
agency.txt not code

josee-sabourin commented on docs/schedule/process.ko.md:
```diff
@@ -0,0 +1,335 @@
+
+  **2012년 10월 15일**<br/>
+
+  추가됨trips.txt 'wheelchair_accessible ' 사양에 대한 제안: [토론](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+   * Se o defensor continuar o trabalho sobre a proposta, então uma nova votação pode ser convocada a qualquer time.
+1. Qualquer solicitação pull que permaneça inativa por 30 dias corridos será encerrada. Quando uma solicitação pull é fechada, a proposta correspondente é considerada abandonada. O defensor pode reabrir o pedido de puxar a qualquer time se desejar continuar ou manter a conversa.
+1. Se a proposta for aceita:
+   * O Google se compromete a fundir a versão votada da solicitação pull (desde que os colaboradores tenham assinado o [CLA](../CONTRIBUTING.md)), e realizar a solicitação pull dentro de 5 dias úteis.
```
"CLA" doesn't correspond with above translation

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**Os feeds devem ser fáceis de analisar**,<br/>os leitores devem ser capazes de extrair as informações que estão procurando com o mínimo de trabalho possível. As mudanças e adições ao feed devem ser o mais amplamente útil possível, para minimizar o número de caminhos de código que os leitores do feed precisam implementar. (Entretanto, deve ser dada precedência à criação, uma vez que no final have ...
+
+**A especificação é sobre informação aos passageirosGTFS** se<br/>preocupa principalmente com a informação aos passageiros. Ou seja, as especificações devem incluir informações que possam ajudar a impulsionar ferramentas para os cavaleiros, em primeiro lugar e acima de tudo. Há potencialmente uma grande quantidade de informações orientadas a operações que as agências de trânsito podem querer tran ...
```
formatting error in title

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**A especificação é sobre informação aos passageirosGTFS** se<br/>preocupa principalmente com a informação aos passageiros. Ou seja, as especificações devem incluir informações que possam ajudar a impulsionar ferramentas para os cavaleiros, em primeiro lugar e acima de tudo. Há potencialmente uma grande quantidade de informações orientadas a operações que as agências de trânsito podem querer tran ...
+
+**Ao**<br/>acrescentar características à especificação, queremos evitar fazer alterações que tornem inválidas as alimentações existentes. Não queremos criar mais trabalho para os editores de feeds existentes até que eles queiram adicionar capacidades a seus feeds. Além disso, sempre que possível, queremos que os analisadores existentes possam continuar a ler as partes mais antigas dos feeds mais  ...
```
translation error in title

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**Ao**<br/>acrescentar características à especificação, queremos evitar fazer alterações que tornem inválidas as alimentações existentes. Não queremos criar mais trabalho para os editores de feeds existentes até que eles queiram adicionar capacidades a seus feeds. Além disso, sempre que possível, queremos que os analisadores existentes possam continuar a ler as partes mais antigas dos feeds mais  ...
+
+As**características especulativas são desencorajadasTudo o**<br/>novo recurso adiciona complexidade à criação e leitura dos feeds. Portanto, queremos ter o cuidado de acrescentar apenas recursos que sabemos serem úteis. Idealmente, qualquer proposta terá sido testada através da geração de dados para um sistema de trânsito real que usa o novo recurso e software de escrita para lê-lo e exibi-lo. No ...
```
formatting error in title

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**25 de maio de 2020**<br/>
+
+* Definidos `pathways.txt`, `levels.txt` e `attributions.txt` como tabelas traduzíveis. Acrescentadas recomendações para a tradução de `valores_como valores_sigalizados` multilíngues. Ver [discussão](https://github.com/google/transit/pull/220).
```
signposted_as translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**13 de maio de 2020**<br/>
+
+* Acrescentadas `a_pickup_pickup_continua` e `drop_off_off_continua` às `routes.txt` e `stop_times.txt`. Mudado o `shape_id` de "Opcional" para "Requerido_condicionalmente". Veja a [discussão](https://github.com/google/transit/pull/208).
```
continuous_pickup, continuous_drop_off translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**5 de fevereiro de 2020**<br/>
+
+* Acrescentado `tts_stop_name` para stops.txt. Ver discussão. Veja a [discussão](https://github.com/google/transit/pull/174).
```
stops.txt not code, but I also think this is completely wrong, on gtfs.org the feb 5 change was "Added trolleybus and monorail `route_types`. See [discussion](https://github.com/google/transit/pull/174)." Unsure if I caught this for the other languages

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**9 de julho de 2019**<br/>
+
+* Adicionadas as melhores práticas de time arrival e departure. Ver [discussão](https://github.com/google/transit/pull/165).
```
i feel like arrival and departure did not get translated here

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**14 de setembro de 2018**<br/>
+
+* Adicionado o conceito "Condicionalmente necessário". Ver [discussão](https://github.com/google/transit/pull/100).
```
conditionally required translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**27 de novembro de 2016**<br/>
+
+* Acrescentado entrada da estação como um `tipo_stop.location_type`. Veja a [discussão](https://github.com/google/transit/pull/30).
```
stop.location_type translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**15 de outubro de 2012**<br/>
+
+Acrescentou trips.txtwheelchair_accessible' proposta para spec: [discussão](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**20 de junho de 2012**<br/>
+
+* Adicionada proposta 'cadeira de rodas_boarding' à especificação: [discussão](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
wheelchair_boarding translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+**6 de setembro de 2011**<br/>
+
+* Adicionada a proposta 'agency_fare_url' à especificação: [discussão](https://groups.google.com/forum/#!topic/gtfs-changes/Zp9rPG07CgE)
+* Adicionada proposta de 'tempos_exatos' à especificação: [discussão](https://groups.google.com/forum/#!topic/gtfs-changes/nZF9lbQ7TQs)
```
exact_times translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+* Esclarecimentos sobre o formato CSV: [discussão](https://groups.google.com/forum/#!topic/gtfs-changes/03qz5aTA2mk).
+* Orientação adicional sobre como escolher cores contrastantes nas descrições dos campos route_color e route_text_color.
+* trip_short_name, como proposto e testado nestes tópicos: a e b.
+* Uma correção para um erro menor nos dados da amostra incluídos no end do documento (dando à parada S7 a estação_mãe S8).
```
parent_station translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+* Adicionados campos do tipo location_type e parent_station aos stops.txt, permitindo que os pontos de parada sejam agrupados em estações[(proposta original](https://groups.google.com/forum/#!topic/gtfs-changes/ScGAyZ9a_yw))
+* Adicionado campo agency_phone para fornecer número de telefone de voz para uma agência[(proposta original](https://groups.google.com/forum/#!topic/gtfs-changes/8Itt58ueyqA))
+* Acrescentada a seção "Testando seus Feeds", mencionando ferramentas de teste de código aberto
+* Esclarecimentos adicionais sobre o formato CSV, zona_horário_da_agência, zona_horário_da_agência, cor_da_hora_da_via_texto, cor_hora_de_chegada, hora_de_partida, calendar.txt vs. calendar_dates.txt, tabelas de tarifas e frequencies.txt
```
agency_timezone, agency_lang, route_color, route_text_color, arrival_time, departure_time translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+* Adicionado o campo stop_code em stops.txt para permitir a especificação de códigos de parada virados para o cavaleiro[(proposta original](https://groups.google.com/forum/#!topic/gtfs-changes/k9A95fYZexc))
+* Esclareceu as descrições de route_short_name e route_long_name em routes.txt
+* Esclareceu as descrições de hora_de\_ chegada e hora_de_partida em stop_times.txt
```
arrival_time and departure_time translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+
+**20 de novembro de 2007**<br/>
+
+* Descrição esclarecida do bloco_id
```
block_id translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+* Adição de URLs por rota e por parada.
+* Adição de campo direction_id em trips.txt
+* Suporte para mudanças de sinal de cabeçalho de viagem com adição de campo stop_headsign em stop_times.txt.
+* Suporte para cores de rota com adição de cor_de_caminho opcional e cor_de_caminho_texto_em routes.txt.
```
route_color and route_text_color translated

josee-sabourin commented on docs/schedule/process.pt_BR.md:
```diff
@@ -0,0 +1,336 @@
+* Suporte para cores de rota com adição de cor_de_caminho opcional e cor_de_caminho_texto_em routes.txt.
+* Removida a capacidade de especificar paradas usando endereços de rua. A versão anterior da especificação permitia dar a localização de uma parada de trânsito usando um endereço de rua nos campos stop_street, stop_city, stop_region, stop_postcode, e stop_country. Agora os locais de parada devem ser dados usando stop_lat para latitude e stop_lon para longitude, que são mais úteis para a maioria d ...
+* Adição do tipo de vehicle de teleférico para o campo route_type em routes.txt
+* Veja o resumo das mudanças no [blog](https://headwayblog.com/2007/03/02/google-feed-spec-update-2007-02/) original do [Headway](https://headwayblog.com/2007/03/02/google-feed-spec-update-2007-02/).
```
fragmented hyperlink

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+**Каналы должны быть просты в создании и редактированииМы**<br/>выбрали CSV в качестве основы для спецификации, потому что его легко просматривать и редактировать с помощью программ электронных таблиц и text редакторов, что полезно для небольших агентств. Его также легко генерировать из большинства языков программирования и баз данных, что полезно для издателей больших фидов.<br/>
+
+**Фиды должны легко**<br/>разбиратьсяЧитатели<br/>**фидов**<br/>должны иметь возможность извлекать нужную им информацию с минимальными затратами. Изменения и дополнения к фиду должны быть как можно более широко полезными, чтобы свести к минимуму количество путей кода, которые читатели фида должны реализовать. (Однако приоритет должен быть отдан упрощению процесса создания, поскольку в конечном ит ...
```
title format error

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+**13 сентября 2021 г.**<br/>
+
+* Обновлены лучшие практики использования `имени_остановки`. См. [обсуждение](https://github.com/google/transit/pull/282).
```
stop_name translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+**5 февраля 2020 года**<br/>
+
+* Добавлены `типы_маршрутов` троллейбуса и монорельса. См. [обсуждение](https://github.com/google/transit/pull/174).
```
route_types translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+**Август 26, 2019**<br/>
+
+* Указано, что `остановки stop_lat` и `stop_lon` должны располагаться там, где пассажиры ожидают посадки в vehicle. См. [обсуждение](https://github.com/google/transit/pull/179).
```
stop_lat code format error

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+**14 сентября 2018 г.**<br/>
+
+* Добавлено понятие "Условно обязательный". См. [обсуждение](https://github.com/google/transit/pull/100).
```
conditionally required translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+**27 ноября 2016 г.**<br/>
+
+* Добавлен вход на станцию как `тип_местоположения_остановки`. См. [обсуждение](https://github.com/google/transit/pull/30).
```
stops.location_type translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+* Уточнения формата CSV: [обсуждение](https://groups.google.com/forum/#!topic/gtfs-changes/03qz5aTA2mk).
+* Дополнительное руководство по выбору контрастных цветов в описаниях полей route_color и route_text_color.
+* trip_short_name, как предложено и проверено в этих темах: a и b.
+* Исправлена незначительная ошибка в примерах данных, включенных в end документа (придание остановке S7 родительской\_станции S8).
```
parent_station translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+**7 августа 2008 г.**<br/>
+
+* Восстановлено поле stop_url, которое было случайно опущено в версии от 6 августа.
+* Добавлен телефон агентства к образцу данных
```
agency_phone translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+* Добавлены поля location_type и parent_station в файл stops.txt, позволяющие группировать остановочные пункты в станции[(первоначальное предложение](https://groups.google.com/forum/#!topic/gtfs-changes/ScGAyZ9a_yw)).
+* Добавлено поле agency_phone для предоставления голосового телефонного номера агентства[(первоначальное предложение](https://groups.google.com/forum/#!topic/gtfs-changes/8Itt58ueyqA))
+* Добавлен раздел "Тестирование ваших фидов" с упоминанием инструментов тестирования с открытым исходным кодом
+* Добавлены пояснения относительно формата CSV, временной зоны агентства, языка агентства, цвета\_маршрута, цвета\_текста\_маршрута, времени\_прибытия, времени\_отправления, calendar.txt против calendar_dates.txt, таблиц тарифов и frequencies.txt
```
agency_timezone, agency_lang, route_color, route_text_color, arrival_time, departure_time translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+**29 февраля 2008 г.**<br/>
+
+* Добавлено поле stop_code в файле stops.txt, позволяющее указывать коды остановок для пассажиров[(первоначальное предложение](https://groups.google.com/forum/#!topic/gtfs-changes/k9A95fYZexc)).
+* Уточнены описания маршрута\_короткое\_имя и маршрута\_длинное\_имя в routes.txt
```
route_short_name and route_long_name translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+* Добавлено поле stop_code в файле stops.txt, позволяющее указывать коды остановок для пассажиров[(первоначальное предложение](https://groups.google.com/forum/#!topic/gtfs-changes/k9A95fYZexc)).
+* Уточнены описания маршрута\_короткое\_имя и маршрута\_длинное\_имя в routes.txt
+* Уточнены описания времени прибытия\_ и времени отправления\_ в файле stop_times.txt
```
arrival_time and departure_time translated

josee-sabourin commented on docs/schedule/process.ru.md:
```diff
@@ -0,0 +1,336 @@
+
+* Добавлена поддержка информации о Shape trip через файл shapes.txt
+* Уточнено определение stop_sequence
+* Пометки тип\_забора и тип\_выгрузки стали необязательными.
```
pickup_type and drop_off_type translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+1. 下面是对该提案的讨论。拉动请求的评论应作为唯一的讨论论坛。
+   * 讨论持续的时间，只要倡导者认为有必要，但必须至少是7个日历日。
+   * 倡导者负责根据他们同意的评论及时更新提案（即拉动请求）。
+   * 在任何time，倡导者都可以声称提案被放弃。
```
time did not get translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+1. 如果提案被接受。
+   * 谷歌承诺合并拉动请求的投票版本（前提是贡献者已经签署了[CLA](../CONTRIBUTING.md)），并在5个工作日内执行拉动请求。
+   * 翻译不得包含在原始拉动请求中。
+   谷歌负责最终将相关翻译更新为支持的语言，但欢迎来自社区的纯Translation拉动请求，一旦所有编辑意见得到解决，就会被接受。
```
translation did not get translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+   * 在投票期间，只允许对提案进行编辑性修改（错别字，只要不改变含义，措辞可以改变）。
+   * 任何人都可以以评论的形式对拉动请求投赞成/反对票，并且在投票期end前可以改变投票。
+   如果一个投票者改变了她的投票，建议通过删除投票内容并写上新的投票来更新原始投票评论。
+   * 投票期start前的投票不予考虑。
```
start did not get translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+1. 投票持续的最短时间足以涵盖7个FULL日历日和5个FULL瑞士工作日。投票在23:59:59 UTC结束。
+   * 倡导者应在投票start时宣布具体的end time。
+   * 在投票期间，只允许对提案进行编辑性修改（错别字，只要不改变含义，措辞可以改变）。
+   * 任何人都可以以评论的形式对拉动请求投赞成/反对票，并且在投票期end前可以改变投票。
```
end did not get translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+   * 在任何time，倡导者都可以声称提案被放弃。
+1. 倡导者可以在讨论所需的最初7天间隔之后的任何time点要求对提案的一个版本进行投票。
+   * 在要求投票之前，至少有一个GTFS生产者和一个GTFS消费者应实施提议的改变。 希望GTFS生产者在面向公众的GTFS馈送中包含该变更，并在拉动请求的评论中提供该数据的链接，而且GTFS消费者在拉动请求的评论中提供一个链接，以非微不足道的方式利用该变更（即支持新的或改进的功能）的应用程序。
+1. 投票持续的最短时间足以涵盖7个FULL日历日和5个FULL瑞士工作日。投票在23:59:59 UTC结束。
```
FULL did not get translated x2

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+1. 倡导者可以在讨论所需的最初7天间隔之后的任何time点要求对提案的一个版本进行投票。
+   * 在要求投票之前，至少有一个GTFS生产者和一个GTFS消费者应实施提议的改变。 希望GTFS生产者在面向公众的GTFS馈送中包含该变更，并在拉动请求的评论中提供该数据的链接，而且GTFS消费者在拉动请求的评论中提供一个链接，以非微不足道的方式利用该变更（即支持新的或改进的功能）的应用程序。
+1. 投票持续的最短时间足以涵盖7个FULL日历日和5个FULL瑞士工作日。投票在23:59:59 UTC结束。
+   * 倡导者应在投票start时宣布具体的end time。
```
start, end time did not get translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+   * 讨论持续的时间，只要倡导者认为有必要，但必须至少是7个日历日。
+   * 倡导者负责根据他们同意的评论及时更新提案（即拉动请求）。
+   * 在任何time，倡导者都可以声称提案被放弃。
+1. 倡导者可以在讨论所需的最初7天间隔之后的任何time点要求对提案的一个版本进行投票。
```
time did not get translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**该规范是关于乘客信息的，GTFS**<br/>主要关注的是乘客信息。也就是说，该规范应该包括能够帮助乘客使用电动工具的信息，这一点是首要的。交通机构可能希望在系统之间内部传输大量的面向运营的信息。GTFS不是为了这个目的，可能有其他面向运营的数据标准可能更合适。<br/>
+
+**对规范的修改应该是向后兼容的当**<br/>向规范添加功能<br/>**时**<br/>，我们希望避免做出会使现有馈送无效的修改。我们不希望为现有的信息源发布者创造更多的工作，直到他们想为他们的信息源增加功能。此外，只要有可能，我们希望现有的分析器能够继续阅读新的饲料的旧部分。<br/>
```
title formatting error

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2021年9月13日**<br/>
+
+* 更新了`stop_name的`最佳做法。见[讨论](https://github.com/google/transit/pull/282)。
```
stop_name format error

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2021年1月4日**<br/>
+
+* 澄清了`stop_times`的描述`。stop_id`。参见[讨论](https://github.com/google/transit/pull/258)。
```
stop_id format error

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2020年5月13日**<br/>
+
+* 在`routes.txt`txt和`stop_times.txt`中增加了`continuous_pickup`和`continuous_drop_off`。将`shape_id`从 "可选 "改为 "有条件要求"。见[讨论](https://github.com/google/transit/pull/208)。
```
optional and conditionally required translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2020年3月24日**<br/>
+
+* 定义了text字段，并在`stops.txt` `txt`中添加了`tts_stop_name`。参见[讨论](https://github.com/google/transit/pull/49)。
```
"text" not translated, extra txt

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2020年2月5日**<br/>
+
+* 增加了无轨电车和单轨电车`的路线类型`。参见[讨论](https://github.com/google/transit/pull/174)。
```
route_types translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2019年7月9日**<br/>
+
+* 增加了arrival和departure time最佳实践。参见[讨论](https://github.com/google/transit/pull/165)。
```
arrival, departure not translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2018年9月14日**<br/>
+
+* 增加了 "有条件要求 "的概念。参见[讨论](https://github.com/google/transit/pull/100)。
```
Conditionally required translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2018年8月22日**<br/>
+
+* 在`feed_info.txt`文件中增加了feed\_`contact_email`和`feed_contact_url`字段。参见[讨论](https://github.com/google/transit/pull/31)。
```
feed_contact_email format error

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2017年12月11日**<br/>
+
+* 在`routes.txt`txt中添加了route`_sort_order`。请参见[讨论](https://github.com/google/transit/pull/83)。
```
route_sort_order format error

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2012年10月15日**<br/>
+
+添加trips.txtwheelchair_accessible' 提案到规范：[讨论](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+**2008年8月7日**<br/>
+
+* 恢复了stop_url字段，该字段在8月6日的版本中被意外地省略了。
+* 在样本数据中增加了机构电话
```
agency_phone translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+* 在stops.txt中增加了location_type和parent_station字段，允许将停车点归入车站[原始提议](https://groups.google.com/forum/#!topic/gtfs-changes/ScGAyZ9a_yw)）。
+* 增加了agency_phone字段，用于提供一个机构的语音电话号码[原始提案](https://groups.google.com/forum/#!topic/gtfs-changes/8Itt58ueyqA)）。
+* 增加了 "测试你的报文 "一节，提到了开源的测试工具
+* 增加了对CSV格式、机构时区、机构语言、路线颜色、路线文本颜色、到达时间、出发时间、calendar.txt与calendar_dates.txt、票价表和frequencies.txt的说明。
```
agency_timezone, agency_lang, route_color, route_text_color, arrival_time, departure_time translated

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+**2008年2月29日**<br/>
+
+* 在stops.txt中增加了 stop_code 字段，允许指定面向乘客的车站代码[原始提案](https://groups.google.com/forum/#!topic/gtfs-changes/k9A95fYZexc)）。
+* 澄清了routesroutes.txttxt中的route_short_name和route_long_name的描述。
```
duplicate routes.txt

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+* 增加了关于[提交信息源](https://developers.google.com/transit/google-transit#SubmitFeedToGoogle)的部分。
+* 增加了[示范交通机构馈送的例子](https://developers.google.com/transit/gtfs/examples/gtfs-feed)。
+* 增加了注意，如果所有的服务日期都在calendar_dates.txt中定义，可以省略calendar.txttxt。
```
extra txt

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+
+**2007年3月8日**<br/>
+
+* 稍作编辑，将stop_url字段从stop_times.txt（在2月28日的更新中，该字段被错误地指定为stop_times.stops.txt）移至stops.txt，即它所属的位置。
```
something is wrong here, gtfs.org does not have brackets in this line, and there seems to be an extra stops.txt?

josee-sabourin commented on docs/schedule/process.zh.md:
```diff
@@ -0,0 +1,336 @@
+* 增加了对票价信息的支持
+* 删除了个别文件名中的日期
+* 改变了route_type值的定义
+* 允许在time发布多个feed文件，只要它们的服务期不重叠。
```
time, feed not translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+   - 在任何時候time倡導者可以要求提案被放棄。
+1. 倡導者可以在任何時候要求對提案的一個版本進行投票time在討論所需的最初 7 天間隔之後。
+   - 在要求投票之前，至少有一個GTFS製片人和一個GTFS消費者應實施提議的變更。預計GTFS生產者包括面向公眾的變化GTFS在拉取請求評論中提供並提供指向該數據的鏈接，並且GTFS消費者在 pull request 評論中提供了一個指向應用程序的鏈接，該應用程序正在以非平凡的方式利用更改（即，它支持新的或改進的功能）。
+1. 投票持續的最短時間足以覆蓋 7FULL日曆日和 5FULL瑞士工作日。投票於世界標準時間 23:59:59 結束。
```
FULL not translated x2

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+1. 倡導者可以在任何時候要求對提案的一個版本進行投票time在討論所需的最初 7 天間隔之後。
+   - 在要求投票之前，至少有一個GTFS製片人和一個GTFS消費者應實施提議的變更。預計GTFS生產者包括面向公眾的變化GTFS在拉取請求評論中提供並提供指向該數據的鏈接，並且GTFS消費者在 pull request 評論中提供了一個指向應用程序的鏈接，該應用程序正在以非平凡的方式利用更改（即，它支持新的或改進的功能）。
+1. 投票持續的最短時間足以覆蓋 7FULL日曆日和 5FULL瑞士工作日。投票於世界標準時間 23:59:59 結束。
+   - 倡導者應公佈具體endtime在start的投票。
```
endtime, start not translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+1. 如果提案被接受：
+   - Google 承諾合併投票通過的拉取請求版本（前提是貢獻者已簽署[CLA](../CONTRIBUTING.md) ），並在 5 個工作日內執行拉取請求。
+   - 翻譯不得包含在原始拉取請求中。
+   Google 負責最終將相關翻譯更新為受支持的語言，但純Translation歡迎來自社區的拉取請求，並在所有編輯評論得到解決後立即接受。
```
Translation not translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2021 年 1 月 4 日**<br/>
+
+* 澄清了`stop_times. stop_id`stop_id .見[討論](https://github.com/google/transit/pull/258)。
```
extra stop_id

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2020 年 5 月 25 日**<br/>
+
+* 定義pathways.txt ,levels.txt和attributions.txt作為可翻譯的表格。添加了翻譯多語言`signposted_as`值的建議。見[討論](https://github.com/google/transit/pull/220)。
```
pathways.txt, levels.txt and attributions.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2020 年 5 月 13 日**<br/>
+
+* 添加了`continuous_pickup`和`continuous_drop_off`到routes.txt和stop_times.txt .改變了shape_id從“可選”到“有條件地需要”。見[討論](https://github.com/google/transit/pull/208)。
```
routes.txt, stop_times.txt, shape_id not code
Optional and conditionally required translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2020 年 3 月 24 日**<br/>
+
+* 定義text-to-speech 字段並將`tts_stop_name`添加到stops.txt .見[討論](https://github.com/google/transit/pull/49)。
```
stops.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2020 年 1 月 9 日**<br/>
+
+* 添加translations.txt .見[討論](https://github.com/google/transit/pull/180)。
```
translations.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2019 年 12 月 26 日**<br/>
+
+* 更新了纜車和空中升降機的定義route_type .見[討論](https://github.com/google/transit/pull/186)。
```
route_type not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2019 年 12 月 20 日**<br/>
+
+* 添加attributions.txt .見[討論](https://github.com/google/transit/pull/192)。
```
attributions.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2019 年 7 月 9 日**<br/>
+
+* 添加arrival和departuretime最佳實踐。見[討論](https://github.com/google/transit/pull/165)。
```
arrival, departure time not trnaslated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+**2019 年 7 月 9 日**<br/>
+
+* 添加arrival和departuretime最佳實踐。見[討論](https://github.com/google/transit/pull/165)。
+* 添加了 headsign 最佳實踐。見[討論](https://github.com/google/transit/pull/167)。
```
headsign not translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+* 添加arrival和departuretime最佳實踐。見[討論](https://github.com/google/transit/pull/165)。
+* 添加了 headsign 最佳實踐。見[討論](https://github.com/google/transit/pull/167)。
+* 添加stop_id最佳實踐。見[討論](https://github.com/google/transit/pull/169)。
```
stop_id not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2019 年 4 月 4 日**<br/>
+
+* 在中添加了`platform_code`字段stops.txt .見[討論](https://github.com/google/transit/pull/146)。
```
stops.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2019 年 3 月 27 日**<br/>
+
+* 添加pathways.txt和levels.txt .見[討論](https://github.com/google/transit/pull/143)。
```
pathways.txt, levels.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2018 年 9 月 14 日**<br/>
+
+* 添加了“有條件地要求”的概念。見[討論](https://github.com/google/transit/pull/100)。
```
conditionally required translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2018 年 8 月 22 日**<br/>
+
+* 添加`feed_contact_email`和`feed_contact_url`字段feed_info.txt文件。見[討論](https://github.com/google/transit/pull/31)。
```
feed_info.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2017 年 12 月 11 日**<br/>
+
+* `route_sort_order`添加到routes.txt .見[討論](https://github.com/google/transit/pull/83)。
```
routes.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+**2017 年 2 月 7 日**<br/>
+
+* 澄清了`block_id`和service_id .見[討論](https://github.com/google/transit/pull/44)。
+* 澄清基於頻率的服務開始於vehicledeparture.見[討論](https://github.com/google/transit/pull/42)。
```
vehicle departure not translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+* 澄清了`block_id`和service_id .見[討論](https://github.com/google/transit/pull/44)。
+* 澄清基於頻率的服務開始於vehicledeparture.見[討論](https://github.com/google/transit/pull/42)。
+* 澄清說明stop_id和`stop_code` 。見[討論](https://github.com/google/transit/pull/40)。
```
stop_id not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2017 年 12 月 11 日**<br/>
+
+* 在中添加`route_sort_order`字段routes.txt文件。見[討論](https://github.com/google/transit/pull/83)。
```
routes.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2016 年 9 月 2 日**<br/>
+
+* 更新文檔以添加agency_id在下面fare_attributes.txt .見[討論](https://github.com/google/transit/pull/27)。
```
agency_id, fare_attributes.txt not code

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+
+**2012 年 10 月 15 日**<br/>
+
+添加trips.txt 'wheelchair_accessible' 規範提案： [討論](https://groups.google.com/forum/?fromgroups#!topic/gtfs-changes/ASxItgsQlh4)
```
missing bullet point

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+* CSV 格式說明： [討論](https://groups.google.com/forum/#!topic/gtfs-changes/03qz5aTA2mk)。
+* 有關如何在 route_color 和 route_text_color 字段的描述中選擇對比色的其他指導。
+* trip_short_name，在這些線程中提出和測試：a 和 b。
+* 修復了包含在示例數據中的一個小錯誤end文檔的（給 stop S7 parent_station S8）。
+* 在示例數據中添加了“agency_lang”信息end正如 Marcy 在評論期間所建議的那樣，該文件的建議： [討論](https://groups.google.com/forum/#!topic/gtfs-changes/5qP1kDUFqx0)。
```
"end" not translated x2

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+* 添加了有關[提交提要的](https://developers.google.com/transit/google-transit#SubmitFeedToGoogle)部分。
+* 添加了[示例演示 Transit Agency feed](https://developers.google.com/transit/gtfs/examples/gtfs-feed) 。
+* 補充說明calendar.txt如果所有服務日期都定義在calendar_dates.txt .
+* 做了agency_id在僅包含一個機構的 Feed 中該字段是可選的。這允許現有的提要沒有agency_id保持有效。
```
Feed not translated

josee-sabourin commented on docs/schedule/process.zh_TW.md:
```diff
@@ -0,0 +1,337 @@
+* 添加了對票價信息的支持
+* 從單個文件名中刪除日期
+* 改變了route_type價值定義
+* 允許同時發布多個提要文件time，只要他們的服務期不重疊
```
time not translated

josee-sabourin commented on docs/schedule/validate.de.md:
```diff
@@ -0,0 +1,45 @@
+<br/>
+
+<div class="usage-buttons">
+   <a class="button" href="https://share.mobilitydata.org/validator-installer-mac">Herunterladen für Mac <img class="icon" src="../../assets/apple.svg" width="20"/></a><a class="button" href="https://share.mobilitydata.org/validator-installer-windows">Herunterladen für Windows <img class="icon" src="../../assets/windows.svg" width="20"/></a><a class="button" href="https://share.mobilitydata.org/v ...
```
icons aren't rendering, likely the case for all languages

josee-sabourin commented on docs/schedule/validate.de.md:
```diff
@@ -0,0 +1,45 @@
+    </div>
+    <div class="usage-video">
+        <video class="center" width="560" height="315" controls="">
+            <source src="../../assets/validator_demo_large.mp4" type="video/mp4">
```
video isn't rendering, likely a problem with all languages

josee-sabourin commented on docs/schedule/validate.fr.md:
```diff
@@ -0,0 +1,45 @@
+# Valideur de GTFS Schedule
+
+Ce validateur d'GTFS Schedule canonique gratuit et open-source est maintenu par [MobilityData](https://mobilitydata.org) avec l'aide de la communauté GTFS. Utilisez-le pour évaluer votre jeu de données par rapport à la [référence](reference.md) officielle [GTFS](reference.md) et aux [meilleures pratiques](best-practices.md). Il fournit un rapport de validation facile à utiliser au format HTML qui ...
```
GTFS reference hyperlink fragmented

josee-sabourin commented on docs/schedule/validate.ja.md:
```diff
@@ -0,0 +1,45 @@
+# GTFSスケジュール検証ツール
+
+このフリーでオープンソースの正規のGTFSスケジュール検証ツールは、GTFSコミュニティの協力を得て、[MobilityDataによって](https://mobilitydata.org)メンテナンスされています。このツールを使って、あなたのデータセットを公式の[GTFSリファレンスと](reference.md) [ベストプラクティスに照らして](best-practices.md)評価することができます。使いやすい検証レポートをHTML形式で提供し、ブラウザで開いて他の関係者と共有することができます。
```
Japanese characters in MobilityData hyperlink

josee-sabourin commented on docs/schedule/validate.ja.md:
```diff
@@ -0,0 +1,45 @@
+
+## データパイプラインの一部としてこのバリデータを使用する
+
+このツールをデータパイプラインで使用する方法の詳細については、[GitHub リポジトリを](https://github.com/MobilityData/gtfs-validator)参照してください。
```
either this link has formatting issues

josee-sabourin commented on docs/schedule/validate.ja.md:
```diff
@@ -0,0 +1,45 @@
+
+このプロジェクトはコミュニティの活動であり、このプロジェクトに興味を持つ人は誰でもコミュニティに参加し、意思決定に参加し、様々な方法でプロジェクトの推進に貢献することができます。新しい機能についてのアイデアを共有する、問題を報告する、ドキュメントを改善する、使い方を教えるなど、さまざまな方法で貢献することができます。
+
+このプロジェクトに貢献することに興味がありますか？[GitHub](https://github.com/MobilityData/gtfs-validator)リポジトリにアクセスし、[貢献のガイドラインを](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md)お読みください。
```
or this one since they are both meant to be "Github repository" translated but are different

josee-sabourin commented on docs/schedule/validate.ko.md:
```diff
@@ -0,0 +1,27 @@
+
+## 이 유효성 검사기를 데이터 파이프라인의 일부로 사용
+
+데이터 파이프라인에서 이 도구를 사용하는 방법에 대한 자세한 지침을 보려면 [를 방문하세요 저장소](https://github.com/MobilityData/gtfs-validator) .
```
same here, either this one is wrong or

josee-sabourin commented on docs/schedule/validate.ko.md:
```diff
@@ -0,0 +1,27 @@
+
+이 프로젝트는 커뮤니티의 노력이며 이 프로젝트에 관심이 있는 사람은 누구나 커뮤니티에 가입하고 의사 결정에 참여하며 다양한 방식으로 프로젝트를 진행하는 데 도움을 줄 수 있습니다. 기여할 수 있는 다양한 방법이 있습니다: 새로운 기능에 대한 아이디어 공유, 문제 보고, 문서 개선, 다른 사람에게 사용 방법 교육 등.
+
+이 프로젝트에 기여하는 데 관심이 있으십니까? [를 방문하십시오 GitHub 저장소](https://github.com/MobilityData/gtfs-validator) 및 [읽기 기여 가이드라인](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md) .
```
this one is wrong, assuming the former since there is not mention of Github in the hyperlink

josee-sabourin commented on docs/schedule/validate.pt_BR.md:
```diff
@@ -0,0 +1,45 @@
+# Validador de GTFS Schedule
+
+Este validador de GTFS Schedule gratuito e de código aberto é mantido pela [MobilityData](https://mobilitydata.org) com a ajuda da comunidade GTFS. Use-o para avaliar seu conjunto de dados em relação à [Referência](reference.md) oficial do [GTFS](reference.md) e às [Melhores Práticas](best-practices.md). Ele fornece um relatório de validação fácil de usar em um formato HTML que pode ser aberto no ...
```
GTFS Reference hyperlink fragmented

josee-sabourin commented on docs/schedule/validate.zh_TW.md:
```diff
@@ -0,0 +1,27 @@
+# GTFS Schedule驗證器
+
+這個免費和開源的規範GTFS Schedule驗證器由[MobilityData](https://mobilitydata.org)在GTFS社區。用它來根據官方評估你的數據集GTFS[參考](reference.md)和[最佳實踐](best-practices.md)。它以 HTML 格式提供易於使用的驗證報告，可以在瀏覽器中打開並與其他方共享。
```
I believe GTFS should be included in the hyperlink to reference.md

josee-sabourin commented on docs/schedule/validate.zh_TW.md:
```diff
@@ -0,0 +1,27 @@
+
+## 使用此驗證器作為數據管道的一部分
+
+要查看有關如何在數據管道中使用此工具的更多說明，請訪問[存儲庫](https://github.com/MobilityData/gtfs-validator)。
```
same issue as other languages, one of these is not like the other!

josee-sabourin commented on docs/schedule/validate.zh_TW.md:
```diff
@@ -0,0 +1,27 @@
+
+這個項目是社區的努力，任何對這個項目感興趣的人都可以加入社區，參與決策並以不同的方式幫助推進它。有許多不同的貢獻方式：分享您對新功能的想法、報告問題、改進文檔、教其他人如何使用它等。
+
+有興趣為這個項目做貢獻嗎？請訪問[GitHub 存儲庫](https://github.com/MobilityData/gtfs-validator)並閱讀我們的[貢獻指南](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md)。
```
If I had to guess this is the right one because it has Github in the link, but what do I know, I am delirious after this review

josee-sabourin reviewed (commented):
done!

emmambd committed:
GTFS Schedule new extensions (#86)

emmambd committed:
feat: extension updates (#87)

* GTFS Schedule new extensions

* escape ASCII bars

github-actions[bot] committed:
[create-pull-request] automated change (#90)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
zh-TW chinese homepage button fix

emmambd committed:
changes page fixes

emmambd committed:
example feed translation fixes

emmambd committed:
attributions example translation fixes

emmambd committed:
continuous stops example fixes

emmambd committed:
fares v1 example fixes

emmambd committed:
fares v2 example fixes

github-actions[bot] committed:
[create-pull-request] automated change (#92)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
fares-v2 fixes

emmambd committed:
feed_info fixes

emmambd committed:
frequencies fixes

emmambd committed:
examples index fixes

emmambd committed:
pathways examples fixes

emmambd committed:
continuous stops add translations

emmambd committed:
frequencies fixes

emmambd committed:
example fixes

emmambd committed:
process translation fixes

emmambd committed:
validate translations

emmambd committed:
translate examples navigation

briandonahue committed:
location_type should be 1 (#93)

emmambd committed:
link fixes

emmambd committed:
feat: Add Fares v2 page (#89)

* fares v2 extension page

* add link to fares v2 on extensions page

* fix first buttons margin

* change official language to adopted

* fares v2 page changes

* updating roadmap link

* update fare containers section

* Fares page translations

* fares v2 translations

github-actions[bot] committed:
[create-pull-request] automated change (#94)

Co-authored-by: maximearmstrong <maximearmstrong@users.noreply.github.com>

emmambd committed:
qa changes

emmambd committed:
text-to-speech translations

emmambd committed:
fares translation fixes

emmambd committed:
changes translations

emmambd committed:
attributions

emmambd committed:
validate page

emmambd committed:
pathways

emmambd committed:
feed info

emmambd committed:
frequencies

emmambd committed:
translations page

isabelle-dr reviewed (approved):
@josee-sabourin and @emmambd this is truly incredible 😍
This PR alone brings so much (so much!) value out there.
Are we going to announce this somewhere? LinkedIn? blogpost?

emmambd commented:
A big shoutout to @josee-sabourin for pitching in on this! 

Since we already announced that we had translations on the website available, I don't think there's any new value for the community in sharing it. (I also still have to do the realtime example PR 😅) 

I will do some brainstorming with Tu-Tho about how we could identify community members who would want to do translation reviews once every 3-6 months, once realtime is wrapped up as well.
